2021-02-16 21:52:48.573403 (MainThread): Running with dbt=0.18.1
2021-02-16 21:52:48.841922 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.rpc.server.RPCServerTask'>, debug=False, defer=None, exclude=None, host='0.0.0.0', log_cache_events=False, log_format='default', models=None, partial_parse=True, port=8580, profile='user', profiles_dir='/usr/src/develop/.dbt', project_dir=None, record_timing_info=None, rpc_method=None, single_threaded=False, state=None, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, use_colors=None, vars='{}', version_check=True, warn_error=False, which='rpc', write_json=True)
2021-02-16 21:52:48.866292 (MainThread): Tracking: tracking
2021-02-16 21:52:48.866487 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc84695c10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc696456d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc68ddd3a0>]}
2021-02-16 21:52:48.866801 (MainThread): Serving RPC server at 0.0.0.0:8580, pid=15
2021-02-16 21:52:48.867204 (MainThread): Supported methods: ['cli_args', 'compile', 'compile_sql', 'deps', 'docs.generate', 'gc', 'get-manifest', 'kill', 'poll', 'ps', 'run', 'run-operation', 'run_sql', 'seed', 'snapshot', 'snapshot-freshness', 'status', 'test']
2021-02-16 21:52:48.867490 (MainThread): Send requests to http://localhost:8580/jsonrpc
2021-02-16 21:52:48.872668 (Thread-1): Parsing macros/grant_select_on_schemas.sql
2021-02-16 21:52:48.876273 (Thread-1): Parsing macros/get_custom_schema.sql
2021-02-16 21:52:48.877282 (Thread-1): Parsing macros/adapters.sql
2021-02-16 21:52:48.905796 (Thread-1): Parsing macros/catalog.sql
2021-02-16 21:52:48.907786 (Thread-1): Parsing macros/materializations/table.sql
2021-02-16 21:52:48.911563 (Thread-1): Parsing macros/materializations/merge.sql
2021-02-16 21:52:48.913489 (Thread-1): Parsing macros/materializations/view.sql
2021-02-16 21:52:48.914926 (Thread-1): Parsing macros/materializations/incremental.sql
2021-02-16 21:52:48.924785 (Thread-1): Parsing macros/core.sql
2021-02-16 21:52:48.928366 (Thread-1): Parsing macros/adapters/common.sql
2021-02-16 21:52:48.972632 (Thread-1): Parsing macros/materializations/helpers.sql
2021-02-16 21:52:48.981237 (Thread-1): Parsing macros/materializations/table/table.sql
2021-02-16 21:52:48.987766 (Thread-1): Parsing macros/materializations/seed/seed.sql
2021-02-16 21:52:49.007772 (Thread-1): Parsing macros/materializations/view/view.sql
2021-02-16 21:52:49.014033 (Thread-1): Parsing macros/materializations/view/create_or_replace_view.sql
2021-02-16 21:52:49.018839 (Thread-1): Parsing macros/materializations/incremental/helpers.sql
2021-02-16 21:52:49.020665 (Thread-1): Parsing macros/materializations/incremental/incremental.sql
2021-02-16 21:52:49.026556 (Thread-1): Parsing macros/materializations/common/merge.sql
2021-02-16 21:52:49.040022 (Thread-1): Parsing macros/materializations/snapshot/strategies.sql
2021-02-16 21:52:49.056981 (Thread-1): Parsing macros/materializations/snapshot/snapshot_merge.sql
2021-02-16 21:52:49.057684 (Thread-2): handling status request
2021-02-16 21:52:49.059713 (Thread-2): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc68dac610>]}
2021-02-16 21:52:49.061420 (Thread-2): sending response (<Response 185 bytes [200 OK]>) to 10.0.12.71
2021-02-16 21:52:49.062122 (Thread-1): Parsing macros/materializations/snapshot/snapshot.sql
2021-02-16 21:52:49.090143 (Thread-1): Parsing macros/schema_tests/relationships.sql
2021-02-16 21:52:49.092104 (Thread-1): Parsing macros/schema_tests/not_null.sql
2021-02-16 21:52:49.093719 (Thread-1): Parsing macros/schema_tests/unique.sql
2021-02-16 21:52:49.095441 (Thread-1): Parsing macros/schema_tests/accepted_values.sql
2021-02-16 21:52:49.098073 (Thread-1): Parsing macros/etc/get_custom_database.sql
2021-02-16 21:52:49.099797 (Thread-1): Parsing macros/etc/query.sql
2021-02-16 21:52:49.100852 (Thread-1): Parsing macros/etc/datetime.sql
2021-02-16 21:52:49.109570 (Thread-1): Parsing macros/etc/get_custom_alias.sql
2021-02-16 21:52:49.110523 (Thread-1): Parsing macros/etc/is_incremental.sql
2021-02-16 21:52:49.112250 (Thread-1): Parsing macros/etc/get_custom_schema.sql
2021-02-16 21:52:49.120192 (Thread-1): * Deprecation Warning: dbt v0.17.0 introduces a new config format for the
dbt_project.yml file. Support for the existing version 1 format will be removed
in a future release of dbt. The following packages are currently configured with
config version 1:
 - jaffleshop

For upgrading instructions, consult the documentation:
  https://docs.getdbt.com/docs/guides/migration-guide/upgrading-to-0-17-0

2021-02-16 21:52:49.120519 (Thread-1): Sending event: {'category': 'dbt', 'action': 'deprecation', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'property_': 'warn', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc68d92520>]}
2021-02-16 21:52:49.168592 (Thread-1): Acquiring new snowflake connection "operation.jaffleshop.jaffleshop-on-run-end-0".
2021-02-16 21:52:49.184341 (Thread-1): Acquiring new snowflake connection "operation.jaffleshop.jaffleshop-on-run-end-1".
2021-02-16 21:52:49.396468 (Thread-1): [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models

2021-02-16 21:52:50.330573 (Thread-3): handling status request
2021-02-16 21:52:50.331000 (Thread-3): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc69686d30>]}
2021-02-16 21:52:50.334403 (Thread-3): sending response (<Response 10753 bytes [200 OK]>) to 10.0.45.116
2021-02-16 21:53:13.097832 (MainThread): Connection 'operation.jaffleshop.jaffleshop-on-run-end-1' was properly closed.
2021-02-16 21:53:13.145614 (Thread-4): Got an acceptable cached parse result
2021-02-16 21:53:13.294968 (Thread-4): [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models

2021-02-16 21:53:13.355983 (Thread-5): handling status request
2021-02-16 21:53:13.356426 (Thread-5): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc6823b850>]}
2021-02-16 21:53:13.357381 (Thread-5): sending response (<Response 852 bytes [200 OK]>) to 10.0.2.84
2021-02-16 21:53:21.295250 (Thread-6): Got an acceptable cached parse result
2021-02-16 21:53:21.452641 (Thread-6): [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models

2021-02-16 21:53:21.454745 (Thread-7): handling status request
2021-02-16 21:53:21.455095 (Thread-7): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc63fa7220>]}
2021-02-16 21:53:21.455982 (Thread-7): sending response (<Response 852 bytes [200 OK]>) to 10.0.26.193
2021-02-16 21:53:30.410325 (Thread-8): Got an acceptable cached parse result
2021-02-16 21:53:30.565694 (Thread-9): handling status request
2021-02-16 21:53:30.566097 (Thread-9): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc6813dbe0>]}
2021-02-16 21:53:30.566787 (Thread-9): sending response (<Response 185 bytes [200 OK]>) to 10.0.28.71
2021-02-16 21:53:30.607325 (Thread-8): [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models

2021-02-16 21:53:31.860863 (Thread-10): handling status request
2021-02-16 21:53:31.861287 (Thread-10): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc682cc850>]}
2021-02-16 21:53:31.862218 (Thread-10): sending response (<Response 852 bytes [200 OK]>) to 10.0.23.61
2021-02-16 21:53:39.321436 (Thread-11): Got an acceptable cached parse result
2021-02-16 21:53:39.507168 (Thread-11): [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models

2021-02-16 21:53:39.521404 (Thread-12): handling status request
2021-02-16 21:53:39.521769 (Thread-12): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc681748b0>]}
2021-02-16 21:53:39.522797 (Thread-12): sending response (<Response 854 bytes [200 OK]>) to 10.0.47.142
2021-02-16 21:54:01.661645 (Thread-13): handling status request
2021-02-16 21:54:01.663363 (Thread-13): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc6808d070>]}
2021-02-16 21:54:01.664258 (Thread-13): sending response (<Response 854 bytes [200 OK]>) to 10.0.4.47
2021-02-16 21:54:07.635255 (Thread-14): Parsing macros/grant_select_on_schemas.sql
2021-02-16 21:54:07.639055 (Thread-14): Parsing macros/get_custom_schema.sql
2021-02-16 21:54:07.640069 (Thread-14): Parsing macros/adapters.sql
2021-02-16 21:54:07.668810 (Thread-14): Parsing macros/catalog.sql
2021-02-16 21:54:07.670798 (Thread-14): Parsing macros/materializations/table.sql
2021-02-16 21:54:07.674659 (Thread-14): Parsing macros/materializations/merge.sql
2021-02-16 21:54:07.676681 (Thread-14): Parsing macros/materializations/view.sql
2021-02-16 21:54:07.678135 (Thread-14): Parsing macros/materializations/incremental.sql
2021-02-16 21:54:07.688080 (Thread-14): Parsing macros/core.sql
2021-02-16 21:54:07.691759 (Thread-14): Parsing macros/adapters/common.sql
2021-02-16 21:54:07.739266 (Thread-14): Parsing macros/materializations/helpers.sql
2021-02-16 21:54:07.748387 (Thread-14): Parsing macros/materializations/table/table.sql
2021-02-16 21:54:07.755008 (Thread-14): Parsing macros/materializations/seed/seed.sql
2021-02-16 21:54:07.777844 (Thread-14): Parsing macros/materializations/view/view.sql
2021-02-16 21:54:07.784118 (Thread-14): Parsing macros/materializations/view/create_or_replace_view.sql
2021-02-16 21:54:07.789081 (Thread-14): Parsing macros/materializations/incremental/helpers.sql
2021-02-16 21:54:07.791278 (Thread-14): Parsing macros/materializations/incremental/incremental.sql
2021-02-16 21:54:07.797200 (Thread-15): handling status request
2021-02-16 21:54:07.798719 (Thread-15): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc63ebe430>]}
2021-02-16 21:54:07.798914 (Thread-14): Parsing macros/materializations/common/merge.sql
2021-02-16 21:54:07.806153 (Thread-15): sending response (<Response 185 bytes [200 OK]>) to 10.0.28.71
2021-02-16 21:54:07.820547 (Thread-14): Parsing macros/materializations/snapshot/strategies.sql
2021-02-16 21:54:07.837251 (Thread-14): Parsing macros/materializations/snapshot/snapshot_merge.sql
2021-02-16 21:54:07.838994 (Thread-14): Parsing macros/materializations/snapshot/snapshot.sql
2021-02-16 21:54:07.866604 (Thread-14): Parsing macros/schema_tests/relationships.sql
2021-02-16 21:54:07.868555 (Thread-14): Parsing macros/schema_tests/not_null.sql
2021-02-16 21:54:07.870086 (Thread-14): Parsing macros/schema_tests/unique.sql
2021-02-16 21:54:07.871815 (Thread-14): Parsing macros/schema_tests/accepted_values.sql
2021-02-16 21:54:07.874431 (Thread-14): Parsing macros/etc/get_custom_database.sql
2021-02-16 21:54:07.876109 (Thread-14): Parsing macros/etc/query.sql
2021-02-16 21:54:07.877161 (Thread-14): Parsing macros/etc/datetime.sql
2021-02-16 21:54:07.885764 (Thread-14): Parsing macros/etc/get_custom_alias.sql
2021-02-16 21:54:07.886717 (Thread-14): Parsing macros/etc/is_incremental.sql
2021-02-16 21:54:07.888308 (Thread-14): Parsing macros/etc/get_custom_schema.sql
2021-02-16 21:54:08.030741 (Thread-14): Acquiring new snowflake connection "operation.jaffleshop.jaffleshop-on-run-end-0".
2021-02-16 21:54:08.036423 (Thread-14): Acquiring new snowflake connection "operation.jaffleshop.jaffleshop-on-run-end-1".
2021-02-16 21:54:08.149940 (Thread-14): [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models

2021-02-16 21:54:09.038068 (Thread-16): handling status request
2021-02-16 21:54:09.038492 (Thread-16): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc63f87d60>]}
2021-02-16 21:54:09.041711 (Thread-16): sending response (<Response 9693 bytes [200 OK]>) to 10.0.23.61
2021-02-16 21:54:24.098869 (Thread-17): handling status request
2021-02-16 21:54:24.099328 (Thread-17): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc63f3a3a0>]}
2021-02-16 21:54:24.102548 (Thread-17): sending response (<Response 9693 bytes [200 OK]>) to 10.0.23.61
2021-02-16 21:54:29.491498 (MainThread): Connection 'operation.jaffleshop.jaffleshop-on-run-end-1' was properly closed.
2021-02-16 21:54:29.517362 (Thread-18): Parsing macros/grant_select_on_schemas.sql
2021-02-16 21:54:29.520775 (Thread-18): Parsing macros/get_custom_schema.sql
2021-02-16 21:54:29.521757 (Thread-18): Parsing macros/adapters.sql
2021-02-16 21:54:29.551174 (Thread-18): Parsing macros/catalog.sql
2021-02-16 21:54:29.553141 (Thread-18): Parsing macros/materializations/table.sql
2021-02-16 21:54:29.556903 (Thread-18): Parsing macros/materializations/merge.sql
2021-02-16 21:54:29.558813 (Thread-18): Parsing macros/materializations/view.sql
2021-02-16 21:54:29.560264 (Thread-18): Parsing macros/materializations/incremental.sql
2021-02-16 21:54:29.570068 (Thread-18): Parsing macros/core.sql
2021-02-16 21:54:29.573615 (Thread-18): Parsing macros/adapters/common.sql
2021-02-16 21:54:29.616564 (Thread-18): Parsing macros/materializations/helpers.sql
2021-02-16 21:54:29.625644 (Thread-18): Parsing macros/materializations/table/table.sql
2021-02-16 21:54:29.632186 (Thread-18): Parsing macros/materializations/seed/seed.sql
2021-02-16 21:54:29.652165 (Thread-18): Parsing macros/materializations/view/view.sql
2021-02-16 21:54:29.658158 (Thread-18): Parsing macros/materializations/view/create_or_replace_view.sql
2021-02-16 21:54:29.662929 (Thread-18): Parsing macros/materializations/incremental/helpers.sql
2021-02-16 21:54:29.668361 (Thread-18): Parsing macros/materializations/incremental/incremental.sql
2021-02-16 21:54:29.674826 (Thread-19): handling status request
2021-02-16 21:54:29.675279 (Thread-18): Parsing macros/materializations/common/merge.sql
2021-02-16 21:54:29.675429 (Thread-19): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc63ece610>]}
2021-02-16 21:54:29.690035 (Thread-19): sending response (<Response 185 bytes [200 OK]>) to 10.0.26.193
2021-02-16 21:54:29.690588 (Thread-18): Parsing macros/materializations/snapshot/strategies.sql
2021-02-16 21:54:29.706816 (Thread-18): Parsing macros/materializations/snapshot/snapshot_merge.sql
2021-02-16 21:54:29.708593 (Thread-18): Parsing macros/materializations/snapshot/snapshot.sql
2021-02-16 21:54:29.736942 (Thread-18): Parsing macros/schema_tests/relationships.sql
2021-02-16 21:54:29.738817 (Thread-18): Parsing macros/schema_tests/not_null.sql
2021-02-16 21:54:29.740347 (Thread-18): Parsing macros/schema_tests/unique.sql
2021-02-16 21:54:29.742028 (Thread-18): Parsing macros/schema_tests/accepted_values.sql
2021-02-16 21:54:29.744658 (Thread-18): Parsing macros/etc/get_custom_database.sql
2021-02-16 21:54:29.746262 (Thread-18): Parsing macros/etc/query.sql
2021-02-16 21:54:29.747659 (Thread-18): Parsing macros/etc/datetime.sql
2021-02-16 21:54:29.756634 (Thread-18): Parsing macros/etc/get_custom_alias.sql
2021-02-16 21:54:29.757619 (Thread-18): Parsing macros/etc/is_incremental.sql
2021-02-16 21:54:29.759487 (Thread-18): Parsing macros/etc/get_custom_schema.sql
2021-02-16 21:54:29.798890 (Thread-18): Acquiring new snowflake connection "operation.jaffleshop.jaffleshop-on-run-end-0".
2021-02-16 21:54:29.804639 (Thread-18): Acquiring new snowflake connection "operation.jaffleshop.jaffleshop-on-run-end-1".
2021-02-16 21:54:29.916954 (Thread-18): [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models

2021-02-16 21:54:31.014882 (Thread-20): handling status request
2021-02-16 21:54:31.015348 (Thread-20): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc63ddb040>]}
2021-02-16 21:54:31.018560 (Thread-20): sending response (<Response 9693 bytes [200 OK]>) to 10.0.47.142
2021-02-16 21:54:43.769257 (Thread-21): handling status request
2021-02-16 21:54:43.769682 (Thread-21): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc63f3a070>]}
2021-02-16 21:54:43.772839 (Thread-21): sending response (<Response 9693 bytes [200 OK]>) to 10.0.40.139
2021-02-16 21:54:48.177968 (Thread-22): handling status request
2021-02-16 21:54:48.178384 (Thread-22): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc63e703a0>]}
2021-02-16 21:54:48.181568 (Thread-22): sending response (<Response 9693 bytes [200 OK]>) to 10.0.34.81
2021-02-16 21:54:53.304475 (MainThread): Connection 'operation.jaffleshop.jaffleshop-on-run-end-1' was properly closed.
2021-02-16 21:54:53.332616 (Thread-23): Parsing macros/grant_select_on_schemas.sql
2021-02-16 21:54:53.336106 (Thread-23): Parsing macros/get_custom_schema.sql
2021-02-16 21:54:53.337194 (Thread-23): Parsing macros/adapters.sql
2021-02-16 21:54:53.365030 (Thread-23): Parsing macros/catalog.sql
2021-02-16 21:54:53.367030 (Thread-23): Parsing macros/materializations/table.sql
2021-02-16 21:54:53.374813 (Thread-23): Parsing macros/materializations/merge.sql
2021-02-16 21:54:53.378043 (Thread-23): Parsing macros/materializations/view.sql
2021-02-16 21:54:53.380375 (Thread-23): Parsing macros/materializations/incremental.sql
2021-02-16 21:54:53.396783 (Thread-23): Parsing macros/core.sql
2021-02-16 21:54:53.402641 (Thread-23): Parsing macros/adapters/common.sql
2021-02-16 21:54:53.456832 (Thread-23): Parsing macros/materializations/helpers.sql
2021-02-16 21:54:53.465856 (Thread-23): Parsing macros/materializations/table/table.sql
2021-02-16 21:54:53.477937 (Thread-23): Parsing macros/materializations/seed/seed.sql
2021-02-16 21:54:53.498580 (Thread-24): handling status request
2021-02-16 21:54:53.498784 (Thread-23): Parsing macros/materializations/view/view.sql
2021-02-16 21:54:53.499212 (Thread-24): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc63ec4130>]}
2021-02-16 21:54:53.506106 (Thread-24): sending response (<Response 185 bytes [200 OK]>) to 10.0.17.191
2021-02-16 21:54:53.506665 (Thread-23): Parsing macros/materializations/view/create_or_replace_view.sql
2021-02-16 21:54:53.511703 (Thread-23): Parsing macros/materializations/incremental/helpers.sql
2021-02-16 21:54:53.513614 (Thread-23): Parsing macros/materializations/incremental/incremental.sql
2021-02-16 21:54:53.520155 (Thread-23): Parsing macros/materializations/common/merge.sql
2021-02-16 21:54:53.539754 (Thread-23): Parsing macros/materializations/snapshot/strategies.sql
2021-02-16 21:54:53.556117 (Thread-23): Parsing macros/materializations/snapshot/snapshot_merge.sql
2021-02-16 21:54:53.557881 (Thread-23): Parsing macros/materializations/snapshot/snapshot.sql
2021-02-16 21:54:53.585721 (Thread-23): Parsing macros/schema_tests/relationships.sql
2021-02-16 21:54:53.587610 (Thread-23): Parsing macros/schema_tests/not_null.sql
2021-02-16 21:54:53.589123 (Thread-23): Parsing macros/schema_tests/unique.sql
2021-02-16 21:54:53.590841 (Thread-23): Parsing macros/schema_tests/accepted_values.sql
2021-02-16 21:54:53.593549 (Thread-23): Parsing macros/etc/get_custom_database.sql
2021-02-16 21:54:53.595207 (Thread-23): Parsing macros/etc/query.sql
2021-02-16 21:54:53.596263 (Thread-23): Parsing macros/etc/datetime.sql
2021-02-16 21:54:53.604911 (Thread-23): Parsing macros/etc/get_custom_alias.sql
2021-02-16 21:54:53.605870 (Thread-23): Parsing macros/etc/is_incremental.sql
2021-02-16 21:54:53.607477 (Thread-23): Parsing macros/etc/get_custom_schema.sql
2021-02-16 21:54:53.646119 (Thread-23): Acquiring new snowflake connection "operation.jaffleshop.jaffleshop-on-run-end-0".
2021-02-16 21:54:53.652352 (Thread-23): Acquiring new snowflake connection "operation.jaffleshop.jaffleshop-on-run-end-1".
2021-02-16 21:54:53.770817 (Thread-23): [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models

2021-02-16 21:54:54.736282 (Thread-25): handling status request
2021-02-16 21:54:54.736709 (Thread-25): Sending event: {'category': 'dbt', 'action': 'rpc_request', 'label': '430d0d8d-8f78-41cc-82f9-66c2d67bcb7c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc63e704f0>]}
2021-02-16 21:54:54.740074 (Thread-25): sending response (<Response 9693 bytes [200 OK]>) to 10.0.1.7
2021-02-16 21:55:56.238808 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc696456d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc84695c10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc68dd78b0>]}
2021-02-16 21:55:56.240924 (MainThread): Flushing usage events
2021-02-16 21:55:56.353798 (MainThread): Connection 'operation.jaffleshop.jaffleshop-on-run-end-1' was properly closed.
2021-02-16 21:55:56.361467 (MainThread): Runtime Error
  at path ['name']: 'dbt-poc' does not match '^[^\\d\\W]\\w*$'

Error encountered in /usr/src/develop/user-16228/environment-20170/repository-16369/dbt_project.yml
2021-02-17 23:25:20.293544 (MainThread): Running with dbt=0.18.0
2021-02-17 23:25:21.890616 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, defer=None, exclude=None, fail_fast=False, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='C:\\Users\\jkolpak\\.dbt', project_dir=None, record_timing_info=None, rpc_method='run', selector_name=None, single_threaded=False, state=None, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, use_colors=None, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2021-02-17 23:25:21.892610 (MainThread): Tracking: tracking
2021-02-17 23:25:21.893765 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000232C5700248>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000232C3550FC8>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000232C35500C8>]}
2021-02-17 23:25:21.921685 (MainThread): Partial parsing not enabled
2021-02-17 23:25:21.923681 (MainThread): Parsing macros\get_custom_schema.sql
2021-02-17 23:25:21.925675 (MainThread): Parsing macros\grant_select_on_schemas.sql
2021-02-17 23:25:21.929665 (MainThread): Parsing macros\adapters.sql
2021-02-17 23:25:21.994331 (MainThread): Parsing macros\catalog.sql
2021-02-17 23:25:21.997322 (MainThread): Parsing macros\materializations\incremental.sql
2021-02-17 23:25:22.011782 (MainThread): Parsing macros\materializations\merge.sql
2021-02-17 23:25:22.015626 (MainThread): Parsing macros\materializations\table.sql
2021-02-17 23:25:22.021610 (MainThread): Parsing macros\materializations\view.sql
2021-02-17 23:25:22.030252 (MainThread): Parsing macros\core.sql
2021-02-17 23:25:22.037072 (MainThread): Parsing macros\adapters\common.sql
2021-02-17 23:25:22.098596 (MainThread): Parsing macros\etc\datetime.sql
2021-02-17 23:25:22.111482 (MainThread): Parsing macros\etc\get_custom_alias.sql
2021-02-17 23:25:22.113961 (MainThread): Parsing macros\etc\get_custom_database.sql
2021-02-17 23:25:22.116949 (MainThread): Parsing macros\etc\get_custom_schema.sql
2021-02-17 23:25:22.119941 (MainThread): Parsing macros\etc\is_incremental.sql
2021-02-17 23:25:22.121936 (MainThread): Parsing macros\etc\query.sql
2021-02-17 23:25:22.123931 (MainThread): Parsing macros\materializations\helpers.sql
2021-02-17 23:25:22.137898 (MainThread): Parsing macros\materializations\common\merge.sql
2021-02-17 23:25:22.157850 (MainThread): Parsing macros\materializations\incremental\helpers.sql
2021-02-17 23:25:22.160842 (MainThread): Parsing macros\materializations\incremental\incremental.sql
2021-02-17 23:25:22.170818 (MainThread): Parsing macros\materializations\seed\seed.sql
2021-02-17 23:25:22.199648 (MainThread): Parsing macros\materializations\snapshot\snapshot.sql
2021-02-17 23:25:22.246757 (MainThread): Parsing macros\materializations\snapshot\snapshot_merge.sql
2021-02-17 23:25:22.249748 (MainThread): Parsing macros\materializations\snapshot\strategies.sql
2021-02-17 23:25:22.273401 (MainThread): Parsing macros\materializations\table\table.sql
2021-02-17 23:25:22.283214 (MainThread): Parsing macros\materializations\view\create_or_replace_view.sql
2021-02-17 23:25:22.290192 (MainThread): Parsing macros\materializations\view\view.sql
2021-02-17 23:25:22.299062 (MainThread): Parsing macros\schema_tests\accepted_values.sql
2021-02-17 23:25:22.303052 (MainThread): Parsing macros\schema_tests\not_null.sql
2021-02-17 23:25:22.305051 (MainThread): Parsing macros\schema_tests\relationships.sql
2021-02-17 23:25:22.308043 (MainThread): Parsing macros\schema_tests\unique.sql
2021-02-17 23:25:22.316022 (MainThread): * Deprecation Warning: dbt v0.17.0 introduces a new config format for the
dbt_project.yml file. Support for the existing version 1 format will be removed
in a future release of dbt. The following packages are currently configured with
config version 1:
 - dbtpoc

For upgrading instructions, consult the documentation:
  https://docs.getdbt.com/docs/guides/migration-guide/upgrading-to-0-17-0

2021-02-17 23:25:22.317019 (MainThread): Sending event: {'category': 'dbt', 'action': 'deprecation', 'label': '51785747-270d-46ef-ac9f-184276eb0b86', 'property_': 'warn', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000232C570E788>]}
2021-02-17 23:25:22.318018 (MainThread): Partial parsing not enabled
2021-02-17 23:25:22.366816 (MainThread): Acquiring new snowflake connection "model.dbtpoc.model".
2021-02-17 23:25:22.393709 (MainThread): Acquiring new snowflake connection "operation.dbtpoc.dbtpoc-on-run-end-0".
2021-02-17 23:25:22.402675 (MainThread): Acquiring new snowflake connection "operation.dbtpoc.dbtpoc-on-run-end-1".
2021-02-17 23:25:22.607655 (MainThread): Found 1 model, 0 tests, 0 snapshots, 0 analyses, 145 macros, 2 operations, 0 seed files, 0 sources
2021-02-17 23:25:22.608652 (MainThread): 
2021-02-17 23:25:22.609647 (MainThread): Acquiring new snowflake connection "master".
2021-02-17 23:25:22.614393 (ThreadPoolExecutor-0_0): Acquiring new snowflake connection "list_DBT_POC".
2021-02-17 23:25:22.634232 (ThreadPoolExecutor-0_0): Using snowflake connection "list_DBT_POC".
2021-02-17 23:25:22.634232 (ThreadPoolExecutor-0_0): On list_DBT_POC: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "list_DBT_POC"} */

    show terse schemas in database DBT_POC
    limit 10000
2021-02-17 23:25:22.635230 (ThreadPoolExecutor-0_0): Opening a new connection, currently in state init
2021-02-17 23:25:24.474837 (ThreadPoolExecutor-0_0): SQL status: SUCCESS 5 in 1.84 seconds
2021-02-17 23:25:24.481362 (ThreadPoolExecutor-0_0): On list_DBT_POC: Close
2021-02-17 23:25:24.756408 (ThreadPoolExecutor-0_0): Acquiring new snowflake connection "create_DBT_POC_dbt_[user_name]".
2021-02-17 23:25:24.756408 (ThreadPoolExecutor-0_0): Acquiring new snowflake connection "create_DBT_POC_dbt_[user_name]".
2021-02-17 23:25:24.756408 (ThreadPoolExecutor-0_0): Creating schema "DBT_POC.dbt_[user_name]"
2021-02-17 23:25:24.762392 (ThreadPoolExecutor-0_0): Using snowflake connection "create_DBT_POC_dbt_[user_name]".
2021-02-17 23:25:24.762392 (ThreadPoolExecutor-0_0): On create_DBT_POC_dbt_[user_name]: BEGIN
2021-02-17 23:25:24.762392 (ThreadPoolExecutor-0_0): Opening a new connection, currently in state closed
2021-02-17 23:25:25.671743 (ThreadPoolExecutor-0_0): SQL status: SUCCESS 1 in 0.91 seconds
2021-02-17 23:25:25.671743 (ThreadPoolExecutor-0_0): Using snowflake connection "create_DBT_POC_dbt_[user_name]".
2021-02-17 23:25:25.671743 (ThreadPoolExecutor-0_0): On create_DBT_POC_dbt_[user_name]: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "create_DBT_POC_dbt_[user_name]"} */
create schema if not exists DBT_POC.dbt_[user_name]
2021-02-17 23:25:25.956772 (ThreadPoolExecutor-0_0): Snowflake query id: 019a5c5d-059e-b34d-0000-40150030207a
2021-02-17 23:25:25.957770 (ThreadPoolExecutor-0_0): Snowflake error: 001003 (42000): SQL compilation error:
syntax error line 1 at position 40 unexpected '['.
2021-02-17 23:25:25.957770 (ThreadPoolExecutor-0_0): Error running SQL: macro create_schema
2021-02-17 23:25:25.957770 (ThreadPoolExecutor-0_0): Rolling back transaction.
2021-02-17 23:25:25.957770 (ThreadPoolExecutor-0_0): On create_DBT_POC_dbt_[user_name]: ROLLBACK
2021-02-17 23:25:26.352928 (ThreadPoolExecutor-0_0): On create_DBT_POC_dbt_[user_name]: Close
2021-02-17 23:25:26.618677 (MainThread): Connection 'master' was properly closed.
2021-02-17 23:25:26.618677 (MainThread): Connection 'create_DBT_POC_dbt_[user_name]' was properly closed.
2021-02-17 23:25:26.618677 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000232C57145C8>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000232C596EE08>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000232C5801C88>]}
2021-02-17 23:25:26.618677 (MainThread): Flushing usage events
2021-02-17 23:25:27.111562 (MainThread): Encountered an error:
2021-02-17 23:25:27.111562 (MainThread): Database Error
  001003 (42000): SQL compilation error:
  syntax error line 1 at position 40 unexpected '['.
2021-02-17 23:25:27.134501 (MainThread): Traceback (most recent call last):
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 161, in exception_handler
    yield
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 78, in add_query
    cursor.execute(sql, bindings)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\cursor.py", line 596, in execute
    errvalue)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\errors.py", line 123, in errorhandler_wrapper
    cursor.errorhandler(connection, cursor, error_class, error_value)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\errors.py", line 88, in default_errorhandler
    done_format_msg=error_value.get('done_format_msg'))
snowflake.connector.errors.ProgrammingError: 001003 (42000): SQL compilation error:
syntax error line 1 at position 40 unexpected '['.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\main.py", line 124, in main
    results, succeeded = handle_and_check(args)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\main.py", line 202, in handle_and_check
    task, res = run_from_args(parsed)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\main.py", line 255, in run_from_args
    results = task.run()
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\runnable.py", line 419, in run
    result = self.execute_with_hooks(selected_uids)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\runnable.py", line 379, in execute_with_hooks
    self.before_run(adapter, selected_uids)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\run.py", line 394, in before_run
    self.create_schemas(adapter, selected_uids)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\runnable.py", line 523, in create_schemas
    create_future.result()
  File "C:\Users\jkolpak\AppData\Local\Continuum\anaconda3\lib\concurrent\futures\_base.py", line 428, in result
    return self.__get_result()
  File "C:\Users\jkolpak\AppData\Local\Continuum\anaconda3\lib\concurrent\futures\_base.py", line 384, in __get_result
    raise self._exception
  File "C:\Users\jkolpak\AppData\Local\Continuum\anaconda3\lib\concurrent\futures\thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\utils.py", line 469, in connected
    return func(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\runnable.py", line 485, in create_schema
    adapter.create_schema(relation)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\impl.py", line 182, in create_schema
    self.execute_macro(CREATE_SCHEMA_MACRO_NAME, kwargs=kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\base\impl.py", line 984, in execute_macro
    result = macro_function(**kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 19, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 28, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 41, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\base\impl.py", line 227, in execute
    fetch=fetch
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 123, in execute
    _, cursor = self.add_query(sql, auto_begin)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 312, in add_query
    abridge_sql_log=abridge_sql_log
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 86, in add_query
    return connection, cursor
  File "C:\Users\jkolpak\AppData\Local\Continuum\anaconda3\lib\contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 178, in exception_handler
    raise DatabaseException(msg)
dbt.exceptions.DatabaseException: Database Error
  001003 (42000): SQL compilation error:
  syntax error line 1 at position 40 unexpected '['.

2021-02-17 23:27:13.180024 (MainThread): Running with dbt=0.18.0
2021-02-17 23:27:14.003350 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, defer=None, exclude=None, fail_fast=False, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='C:\\Users\\jkolpak\\.dbt', project_dir=None, record_timing_info=None, rpc_method='run', selector_name=None, single_threaded=False, state=None, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, use_colors=None, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2021-02-17 23:27:14.004334 (MainThread): Tracking: tracking
2021-02-17 23:27:14.004334 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021E299E0C88>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021E2DA7F388>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021E2DA7FFC8>]}
2021-02-17 23:27:14.030218 (MainThread): Partial parsing not enabled
2021-02-17 23:27:14.031216 (MainThread): Parsing macros\get_custom_schema.sql
2021-02-17 23:27:14.032213 (MainThread): Parsing macros\grant_select_on_schemas.sql
2021-02-17 23:27:14.035205 (MainThread): Parsing macros\adapters.sql
2021-02-17 23:27:14.074195 (MainThread): Parsing macros\catalog.sql
2021-02-17 23:27:14.077135 (MainThread): Parsing macros\materializations\incremental.sql
2021-02-17 23:27:14.093090 (MainThread): Parsing macros\materializations\merge.sql
2021-02-17 23:27:14.095084 (MainThread): Parsing macros\materializations\table.sql
2021-02-17 23:27:14.100072 (MainThread): Parsing macros\materializations\view.sql
2021-02-17 23:27:14.105058 (MainThread): Parsing macros\core.sql
2021-02-17 23:27:14.109047 (MainThread): Parsing macros\adapters\common.sql
2021-02-17 23:27:14.162895 (MainThread): Parsing macros\etc\datetime.sql
2021-02-17 23:27:14.172907 (MainThread): Parsing macros\etc\get_custom_alias.sql
2021-02-17 23:27:14.174861 (MainThread): Parsing macros\etc\get_custom_database.sql
2021-02-17 23:27:14.176885 (MainThread): Parsing macros\etc\get_custom_schema.sql
2021-02-17 23:27:14.178883 (MainThread): Parsing macros\etc\is_incremental.sql
2021-02-17 23:27:14.180875 (MainThread): Parsing macros\etc\query.sql
2021-02-17 23:27:14.182870 (MainThread): Parsing macros\materializations\helpers.sql
2021-02-17 23:27:14.193466 (MainThread): Parsing macros\materializations\common\merge.sql
2021-02-17 23:27:14.210483 (MainThread): Parsing macros\materializations\incremental\helpers.sql
2021-02-17 23:27:14.213485 (MainThread): Parsing macros\materializations\incremental\incremental.sql
2021-02-17 23:27:14.220491 (MainThread): Parsing macros\materializations\seed\seed.sql
2021-02-17 23:27:14.246067 (MainThread): Parsing macros\materializations\snapshot\snapshot.sql
2021-02-17 23:27:14.282868 (MainThread): Parsing macros\materializations\snapshot\snapshot_merge.sql
2021-02-17 23:27:14.285484 (MainThread): Parsing macros\materializations\snapshot\strategies.sql
2021-02-17 23:27:14.305059 (MainThread): Parsing macros\materializations\table\table.sql
2021-02-17 23:27:14.313986 (MainThread): Parsing macros\materializations\view\create_or_replace_view.sql
2021-02-17 23:27:14.319992 (MainThread): Parsing macros\materializations\view\view.sql
2021-02-17 23:27:14.327971 (MainThread): Parsing macros\schema_tests\accepted_values.sql
2021-02-17 23:27:14.330969 (MainThread): Parsing macros\schema_tests\not_null.sql
2021-02-17 23:27:14.332959 (MainThread): Parsing macros\schema_tests\relationships.sql
2021-02-17 23:27:14.334868 (MainThread): Parsing macros\schema_tests\unique.sql
2021-02-17 23:27:14.342875 (MainThread): * Deprecation Warning: dbt v0.17.0 introduces a new config format for the
dbt_project.yml file. Support for the existing version 1 format will be removed
in a future release of dbt. The following packages are currently configured with
config version 1:
 - dbtpoc

For upgrading instructions, consult the documentation:
  https://docs.getdbt.com/docs/guides/migration-guide/upgrading-to-0-17-0

2021-02-17 23:27:14.342875 (MainThread): Sending event: {'category': 'dbt', 'action': 'deprecation', 'label': 'ff6fc46f-ed75-404a-8738-c6b23e14effe', 'property_': 'warn', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021E2DACB088>]}
2021-02-17 23:27:14.344318 (MainThread): Partial parsing not enabled
2021-02-17 23:27:14.393709 (MainThread): Acquiring new snowflake connection "model.dbtpoc.model".
2021-02-17 23:27:14.418642 (MainThread): Acquiring new snowflake connection "operation.dbtpoc.dbtpoc-on-run-end-0".
2021-02-17 23:27:14.426621 (MainThread): Acquiring new snowflake connection "operation.dbtpoc.dbtpoc-on-run-end-1".
2021-02-17 23:27:14.637959 (MainThread): Found 1 model, 0 tests, 0 snapshots, 0 analyses, 145 macros, 2 operations, 0 seed files, 0 sources
2021-02-17 23:27:14.638956 (MainThread): 
2021-02-17 23:27:14.639954 (MainThread): Acquiring new snowflake connection "master".
2021-02-17 23:27:14.644993 (ThreadPoolExecutor-0_0): Acquiring new snowflake connection "list_DBT_POC".
2021-02-17 23:27:14.660986 (ThreadPoolExecutor-0_0): Using snowflake connection "list_DBT_POC".
2021-02-17 23:27:14.660986 (ThreadPoolExecutor-0_0): On list_DBT_POC: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "list_DBT_POC"} */

    show terse schemas in database DBT_POC
    limit 10000
2021-02-17 23:27:14.660986 (ThreadPoolExecutor-0_0): Opening a new connection, currently in state init
2021-02-17 23:27:15.767033 (ThreadPoolExecutor-0_0): SQL status: SUCCESS 5 in 1.11 seconds
2021-02-17 23:27:15.772049 (ThreadPoolExecutor-0_0): On list_DBT_POC: Close
2021-02-17 23:27:16.112804 (ThreadPoolExecutor-1_0): Acquiring new snowflake connection "list_DBT_POC_dbt_jkolpak".
2021-02-17 23:27:16.120753 (ThreadPoolExecutor-1_0): Using snowflake connection "list_DBT_POC_dbt_jkolpak".
2021-02-17 23:27:16.120753 (ThreadPoolExecutor-1_0): On list_DBT_POC_dbt_jkolpak: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "list_DBT_POC_dbt_jkolpak"} */

    show terse objects in DBT_POC.dbt_jkolpak
2021-02-17 23:27:16.120753 (ThreadPoolExecutor-1_0): Opening a new connection, currently in state init
2021-02-17 23:27:17.113887 (ThreadPoolExecutor-1_0): SQL status: SUCCESS 5 in 0.99 seconds
2021-02-17 23:27:17.119872 (ThreadPoolExecutor-1_0): On list_DBT_POC_dbt_jkolpak: Close
2021-02-17 23:27:17.392951 (MainThread): Using snowflake connection "master".
2021-02-17 23:27:17.392951 (MainThread): On master: BEGIN
2021-02-17 23:27:17.392951 (MainThread): Opening a new connection, currently in state init
2021-02-17 23:27:18.225673 (MainThread): SQL status: SUCCESS 1 in 0.83 seconds
2021-02-17 23:27:18.226671 (MainThread): On master: COMMIT
2021-02-17 23:27:18.226671 (MainThread): Using snowflake connection "master".
2021-02-17 23:27:18.226671 (MainThread): On master: COMMIT
2021-02-17 23:27:18.660044 (MainThread): SQL status: SUCCESS 1 in 0.43 seconds
2021-02-17 23:27:18.660044 (MainThread): On master: Close
2021-02-17 23:27:18.992000 (MainThread): 17:27:18 | Concurrency: 16 threads (target='dev')
2021-02-17 23:27:18.992000 (MainThread): 17:27:18 | 
2021-02-17 23:27:18.998981 (Thread-1): Began running node model.dbtpoc.model
2021-02-17 23:27:18.999978 (Thread-1): 17:27:19 | 1 of 1 START view model dbt_jkolpak.model............................ [RUN]
2021-02-17 23:27:19.000976 (Thread-1): Acquiring new snowflake connection "model.dbtpoc.model".
2021-02-17 23:27:19.000976 (Thread-1): Compiling model.dbtpoc.model
2021-02-17 23:27:19.015936 (Thread-1): Writing injected SQL for node "model.dbtpoc.model"
2021-02-17 23:27:19.019930 (Thread-1): finished collecting timing info
2021-02-17 23:27:19.055830 (Thread-1): Writing runtime SQL for node "model.dbtpoc.model"
2021-02-17 23:27:19.059819 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-17 23:27:19.059819 (Thread-1): On model.dbtpoc.model: BEGIN
2021-02-17 23:27:19.059819 (Thread-1): Opening a new connection, currently in state init
2021-02-17 23:27:20.087494 (Thread-1): SQL status: SUCCESS 1 in 1.03 seconds
2021-02-17 23:27:20.088491 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-17 23:27:20.088491 (Thread-1): On model.dbtpoc.model: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "node_id": "model.dbtpoc.model"} */

  create or replace  view DBT_POC.dbt_jkolpak.model  as (
    SELECT * FROM table_taco
  );
2021-02-17 23:27:20.328052 (Thread-1): Snowflake query id: 019a5c5f-05f6-304f-0000-40150030302a
2021-02-17 23:27:20.329043 (Thread-1): Snowflake error: 002003 (42S02): SQL compilation error:
Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
2021-02-17 23:27:20.329043 (Thread-1): finished collecting timing info
2021-02-17 23:27:20.329043 (Thread-1): On model.dbtpoc.model: ROLLBACK
2021-02-17 23:27:20.667211 (Thread-1): On model.dbtpoc.model: Close
2021-02-17 23:27:20.955108 (Thread-1): Database Error in model model (models\model.sql)
  002003 (42S02): SQL compilation error:
  Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
  compiled SQL at target\run\dbtpoc\models\model.sql
Traceback (most recent call last):
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 161, in exception_handler
    yield
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 78, in add_query
    cursor.execute(sql, bindings)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\cursor.py", line 596, in execute
    errvalue)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\errors.py", line 123, in errorhandler_wrapper
    cursor.errorhandler(connection, cursor, error_class, error_value)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\errors.py", line 88, in default_errorhandler
    done_format_msg=error_value.get('done_format_msg'))
snowflake.connector.errors.ProgrammingError: 002003 (42S02): SQL compilation error:
Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\base.py", line 333, in safe_run
    result = self.compile_and_execute(manifest, ctx)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\base.py", line 276, in compile_and_execute
    result = self.run(ctx.node, manifest)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\base.py", line 378, in run
    return self.execute(compiled_node, manifest)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\run.py", line 233, in execute
    result = MacroGenerator(materialization_macro, context)()
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 27, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 65, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 41, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\base\impl.py", line 227, in execute
    fetch=fetch
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 123, in execute
    _, cursor = self.add_query(sql, auto_begin)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 312, in add_query
    abridge_sql_log=abridge_sql_log
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 86, in add_query
    return connection, cursor
  File "C:\Users\jkolpak\AppData\Local\Continuum\anaconda3\lib\contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 178, in exception_handler
    raise DatabaseException(msg)
dbt.exceptions.DatabaseException: Database Error in model model (models\model.sql)
  002003 (42S02): SQL compilation error:
  Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
  compiled SQL at target\run\dbtpoc\models\model.sql
2021-02-17 23:27:20.960095 (Thread-1): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': 'ff6fc46f-ed75-404a-8738-c6b23e14effe', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021E2DCA2688>]}
2021-02-17 23:27:20.961092 (Thread-1): 17:27:20 | 1 of 1 ERROR creating view model dbt_jkolpak.model................... [ERROR in 1.96s]
2021-02-17 23:27:20.961092 (Thread-1): Finished running node model.dbtpoc.model
2021-02-17 23:27:21.018144 (MainThread): Acquiring new snowflake connection "master".
2021-02-17 23:27:21.018144 (MainThread): Using snowflake connection "master".
2021-02-17 23:27:21.018144 (MainThread): On master: BEGIN
2021-02-17 23:27:21.018144 (MainThread): Opening a new connection, currently in state closed
2021-02-17 23:27:21.927756 (MainThread): SQL status: SUCCESS 1 in 0.91 seconds
2021-02-17 23:27:21.927756 (MainThread): On master: COMMIT
2021-02-17 23:27:21.927756 (MainThread): Using snowflake connection "master".
2021-02-17 23:27:21.927756 (MainThread): On master: COMMIT
2021-02-17 23:27:22.248225 (MainThread): SQL status: SUCCESS 1 in 0.32 seconds
2021-02-17 23:27:22.248225 (MainThread): 17:27:22 | 
2021-02-17 23:27:22.249197 (MainThread): 17:27:22 | Running 2 on-run-end hooks
2021-02-17 23:27:22.250196 (MainThread): Compiling operation.dbtpoc.dbtpoc-on-run-end-0
2021-02-17 23:27:22.261165 (MainThread): Writing injected SQL for node "operation.dbtpoc.dbtpoc-on-run-end-0"
2021-02-17 23:27:22.266184 (MainThread): 17:27:22 | 1 of 2 START hook: dbtpoc.on-run-end.0............................... [RUN]
2021-02-17 23:27:22.267179 (MainThread): 17:27:22 | 1 of 2 OK hook: dbtpoc.on-run-end.0.................................. [OK in 0.00s]
2021-02-17 23:27:22.270145 (MainThread): Compiling operation.dbtpoc.dbtpoc-on-run-end-1
2021-02-17 23:27:22.277157 (MainThread): Writing injected SQL for node "operation.dbtpoc.dbtpoc-on-run-end-1"
2021-02-17 23:27:22.279118 (MainThread): 17:27:22 | 2 of 2 START hook: dbtpoc.on-run-end.1............................... [RUN]
2021-02-17 23:27:22.279118 (MainThread): 17:27:22 | 2 of 2 OK hook: dbtpoc.on-run-end.1.................................. [OK in 0.00s]
2021-02-17 23:27:22.280114 (MainThread): 17:27:22 | 
2021-02-17 23:27:22.290134 (MainThread): On master: Close
2021-02-17 23:27:22.525091 (MainThread): 17:27:22 | 
2021-02-17 23:27:22.525091 (MainThread): 17:27:22 | Finished running 1 view model, 2 hooks in 7.89s.
2021-02-17 23:27:22.526090 (MainThread): Connection 'master' was properly closed.
2021-02-17 23:27:22.526090 (MainThread): Connection 'list_DBT_POC' was properly closed.
2021-02-17 23:27:22.527087 (MainThread): Connection 'list_DBT_POC_dbt_jkolpak' was properly closed.
2021-02-17 23:27:22.527087 (MainThread): Connection 'model.dbtpoc.model' was properly closed.
2021-02-17 23:27:22.531078 (MainThread): 
2021-02-17 23:27:22.531078 (MainThread): Completed with 1 error and 0 warnings:
2021-02-17 23:27:22.532074 (MainThread): 
2021-02-17 23:27:22.533072 (MainThread): Database Error in model model (models\model.sql)
2021-02-17 23:27:22.534068 (MainThread):   002003 (42S02): SQL compilation error:
2021-02-17 23:27:22.536091 (MainThread):   Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
2021-02-17 23:27:22.537071 (MainThread):   compiled SQL at target\run\dbtpoc\models\model.sql
2021-02-17 23:27:22.540079 (MainThread): 
Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
2021-02-17 23:27:22.541049 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021E2DF1C848>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021E2DE873C8>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000021E2DCC4848>]}
2021-02-17 23:27:22.541049 (MainThread): Flushing usage events
2021-02-17 23:39:58.230782 (MainThread): Running with dbt=0.18.0
2021-02-17 23:39:59.916005 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, defer=None, exclude=None, fail_fast=False, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='C:\\Users\\jkolpak\\.dbt', project_dir=None, record_timing_info=None, rpc_method='run', selector_name=None, single_threaded=False, state=None, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, use_colors=None, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2021-02-17 23:39:59.918000 (MainThread): Tracking: tracking
2021-02-17 23:39:59.918000 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000138176B0DC8>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001381B71F3C8>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001381B71F388>]}
2021-02-17 23:39:59.949950 (MainThread): Partial parsing not enabled
2021-02-17 23:39:59.951911 (MainThread): Parsing macros\get_custom_schema.sql
2021-02-17 23:39:59.952909 (MainThread): Parsing macros\grant_select_on_schemas.sql
2021-02-17 23:39:59.956895 (MainThread): Parsing macros\adapters.sql
2021-02-17 23:40:00.030698 (MainThread): Parsing macros\catalog.sql
2021-02-17 23:40:00.035686 (MainThread): Parsing macros\materializations\incremental.sql
2021-02-17 23:40:00.051643 (MainThread): Parsing macros\materializations\merge.sql
2021-02-17 23:40:00.056629 (MainThread): Parsing macros\materializations\table.sql
2021-02-17 23:40:00.066602 (MainThread): Parsing macros\materializations\view.sql
2021-02-17 23:40:00.079568 (MainThread): Parsing macros\core.sql
2021-02-17 23:40:00.086550 (MainThread): Parsing macros\adapters\common.sql
2021-02-17 23:40:00.159355 (MainThread): Parsing macros\etc\datetime.sql
2021-02-17 23:40:00.172320 (MainThread): Parsing macros\etc\get_custom_alias.sql
2021-02-17 23:40:00.175312 (MainThread): Parsing macros\etc\get_custom_database.sql
2021-02-17 23:40:00.182294 (MainThread): Parsing macros\etc\get_custom_schema.sql
2021-02-17 23:40:00.191269 (MainThread): Parsing macros\etc\is_incremental.sql
2021-02-17 23:40:00.199248 (MainThread): Parsing macros\etc\query.sql
2021-02-17 23:40:00.204235 (MainThread): Parsing macros\materializations\helpers.sql
2021-02-17 23:40:00.232161 (MainThread): Parsing macros\materializations\common\merge.sql
2021-02-17 23:40:00.264076 (MainThread): Parsing macros\materializations\incremental\helpers.sql
2021-02-17 23:40:00.271056 (MainThread): Parsing macros\materializations\incremental\incremental.sql
2021-02-17 23:40:00.292001 (MainThread): Parsing macros\materializations\seed\seed.sql
2021-02-17 23:40:00.350843 (MainThread): Parsing macros\materializations\snapshot\snapshot.sql
2021-02-17 23:40:00.421654 (MainThread): Parsing macros\materializations\snapshot\snapshot_merge.sql
2021-02-17 23:40:00.427638 (MainThread): Parsing macros\materializations\snapshot\strategies.sql
2021-02-17 23:40:00.460550 (MainThread): Parsing macros\materializations\table\table.sql
2021-02-17 23:40:00.480496 (MainThread): Parsing macros\materializations\view\create_or_replace_view.sql
2021-02-17 23:40:00.499448 (MainThread): Parsing macros\materializations\view\view.sql
2021-02-17 23:40:00.514406 (MainThread): Parsing macros\schema_tests\accepted_values.sql
2021-02-17 23:40:00.520390 (MainThread): Parsing macros\schema_tests\not_null.sql
2021-02-17 23:40:00.525378 (MainThread): Parsing macros\schema_tests\relationships.sql
2021-02-17 23:40:00.530363 (MainThread): Parsing macros\schema_tests\unique.sql
2021-02-17 23:40:00.542332 (MainThread): * Deprecation Warning: dbt v0.17.0 introduces a new config format for the
dbt_project.yml file. Support for the existing version 1 format will be removed
in a future release of dbt. The following packages are currently configured with
config version 1:
 - dbtpoc

For upgrading instructions, consult the documentation:
  https://docs.getdbt.com/docs/guides/migration-guide/upgrading-to-0-17-0

2021-02-17 23:40:00.543329 (MainThread): Sending event: {'category': 'dbt', 'action': 'deprecation', 'label': '767539bf-0858-43ac-976c-95bc56efad5c', 'property_': 'warn', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001381B752888>]}
2021-02-17 23:40:00.545323 (MainThread): Partial parsing not enabled
2021-02-17 23:40:00.637078 (MainThread): Acquiring new snowflake connection "model.dbtpoc.model".
2021-02-17 23:40:00.683953 (MainThread): Acquiring new snowflake connection "operation.dbtpoc.dbtpoc-on-run-end-0".
2021-02-17 23:40:00.695921 (MainThread): Acquiring new snowflake connection "operation.dbtpoc.dbtpoc-on-run-end-1".
2021-02-17 23:40:00.996163 (MainThread): Found 1 model, 0 tests, 0 snapshots, 0 analyses, 145 macros, 2 operations, 0 seed files, 0 sources
2021-02-17 23:40:00.998151 (MainThread): 
2021-02-17 23:40:00.999147 (MainThread): Acquiring new snowflake connection "master".
2021-02-17 23:40:01.004134 (ThreadPoolExecutor-0_0): Acquiring new snowflake connection "list_DBT_POC".
2021-02-17 23:40:01.040040 (ThreadPoolExecutor-0_0): Using snowflake connection "list_DBT_POC".
2021-02-17 23:40:01.041034 (ThreadPoolExecutor-0_0): On list_DBT_POC: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "list_DBT_POC"} */

    show terse schemas in database DBT_POC
    limit 10000
2021-02-17 23:40:01.041034 (ThreadPoolExecutor-0_0): Opening a new connection, currently in state init
2021-02-17 23:40:02.483900 (ThreadPoolExecutor-0_0): SQL status: SUCCESS 5 in 1.44 seconds
2021-02-17 23:40:02.499372 (ThreadPoolExecutor-0_0): On list_DBT_POC: Close
2021-02-17 23:40:02.821510 (ThreadPoolExecutor-1_0): Acquiring new snowflake connection "list_DBT_POC_dbt_jkolpak".
2021-02-17 23:40:02.846443 (ThreadPoolExecutor-1_0): Using snowflake connection "list_DBT_POC_dbt_jkolpak".
2021-02-17 23:40:02.847441 (ThreadPoolExecutor-1_0): On list_DBT_POC_dbt_jkolpak: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "list_DBT_POC_dbt_jkolpak"} */

    show terse objects in DBT_POC.dbt_jkolpak
2021-02-17 23:40:02.847441 (ThreadPoolExecutor-1_0): Opening a new connection, currently in state init
2021-02-17 23:40:04.888700 (ThreadPoolExecutor-1_0): SQL status: SUCCESS 5 in 2.04 seconds
2021-02-17 23:40:04.893653 (ThreadPoolExecutor-1_0): On list_DBT_POC_dbt_jkolpak: Close
2021-02-17 23:40:05.207814 (MainThread): Using snowflake connection "master".
2021-02-17 23:40:05.207814 (MainThread): On master: BEGIN
2021-02-17 23:40:05.208809 (MainThread): Opening a new connection, currently in state init
2021-02-17 23:40:06.474590 (MainThread): SQL status: SUCCESS 1 in 1.27 seconds
2021-02-17 23:40:06.474590 (MainThread): On master: COMMIT
2021-02-17 23:40:06.474590 (MainThread): Using snowflake connection "master".
2021-02-17 23:40:06.474590 (MainThread): On master: COMMIT
2021-02-17 23:40:07.094770 (MainThread): SQL status: SUCCESS 1 in 0.62 seconds
2021-02-17 23:40:07.095766 (MainThread): On master: Close
2021-02-17 23:40:07.507676 (MainThread): 17:40:07 | Concurrency: 16 threads (target='dev')
2021-02-17 23:40:07.508672 (MainThread): 17:40:07 | 
2021-02-17 23:40:07.521640 (Thread-1): Began running node model.dbtpoc.model
2021-02-17 23:40:07.524631 (Thread-1): 17:40:07 | 1 of 1 START view model dbt_jkolpak.model............................ [RUN]
2021-02-17 23:40:07.526627 (Thread-1): Acquiring new snowflake connection "model.dbtpoc.model".
2021-02-17 23:40:07.526627 (Thread-1): Compiling model.dbtpoc.model
2021-02-17 23:40:07.574499 (Thread-1): Writing injected SQL for node "model.dbtpoc.model"
2021-02-17 23:40:07.576492 (Thread-1): finished collecting timing info
2021-02-17 23:40:07.633340 (Thread-1): Writing runtime SQL for node "model.dbtpoc.model"
2021-02-17 23:40:07.635335 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-17 23:40:07.635335 (Thread-1): On model.dbtpoc.model: BEGIN
2021-02-17 23:40:07.635335 (Thread-1): Opening a new connection, currently in state init
2021-02-17 23:40:10.241129 (Thread-1): SQL status: SUCCESS 1 in 2.61 seconds
2021-02-17 23:40:10.241129 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-17 23:40:10.241129 (Thread-1): On model.dbtpoc.model: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "node_id": "model.dbtpoc.model"} */

  create or replace  view DBT_POC.dbt_jkolpak.model  as (
    SELECT * FROM table_taco
  );
2021-02-17 23:40:11.062106 (Thread-1): Snowflake query id: 019a5c6c-0542-5577-0000-40150030303e
2021-02-17 23:40:11.062106 (Thread-1): Snowflake error: 002003 (42S02): SQL compilation error:
Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
2021-02-17 23:40:11.062106 (Thread-1): finished collecting timing info
2021-02-17 23:40:11.063104 (Thread-1): On model.dbtpoc.model: ROLLBACK
2021-02-17 23:40:13.502474 (Thread-1): On model.dbtpoc.model: Close
2021-02-17 23:40:14.770104 (Thread-1): Database Error in model model (models\model.sql)
  002003 (42S02): SQL compilation error:
  Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
  compiled SQL at target\run\dbtpoc\models\model.sql
Traceback (most recent call last):
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 161, in exception_handler
    yield
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 78, in add_query
    cursor.execute(sql, bindings)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\cursor.py", line 596, in execute
    errvalue)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\errors.py", line 123, in errorhandler_wrapper
    cursor.errorhandler(connection, cursor, error_class, error_value)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\errors.py", line 88, in default_errorhandler
    done_format_msg=error_value.get('done_format_msg'))
snowflake.connector.errors.ProgrammingError: 002003 (42S02): SQL compilation error:
Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\base.py", line 333, in safe_run
    result = self.compile_and_execute(manifest, ctx)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\base.py", line 276, in compile_and_execute
    result = self.run(ctx.node, manifest)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\base.py", line 378, in run
    return self.execute(compiled_node, manifest)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\run.py", line 233, in execute
    result = MacroGenerator(materialization_macro, context)()
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 27, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 65, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 41, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\base\impl.py", line 227, in execute
    fetch=fetch
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 123, in execute
    _, cursor = self.add_query(sql, auto_begin)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 312, in add_query
    abridge_sql_log=abridge_sql_log
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 86, in add_query
    return connection, cursor
  File "C:\Users\jkolpak\AppData\Local\Continuum\anaconda3\lib\contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 178, in exception_handler
    raise DatabaseException(msg)
dbt.exceptions.DatabaseException: Database Error in model model (models\model.sql)
  002003 (42S02): SQL compilation error:
  Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
  compiled SQL at target\run\dbtpoc\models\model.sql
2021-02-17 23:40:14.794762 (Thread-1): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': '767539bf-0858-43ac-976c-95bc56efad5c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001381BA1C988>]}
2021-02-17 23:40:14.797754 (Thread-1): 17:40:14 | 1 of 1 ERROR creating view model dbt_jkolpak.model................... [ERROR in 7.27s]
2021-02-17 23:40:14.798751 (Thread-1): Finished running node model.dbtpoc.model
2021-02-17 23:40:14.899484 (MainThread): Acquiring new snowflake connection "master".
2021-02-17 23:40:14.899484 (MainThread): Using snowflake connection "master".
2021-02-17 23:40:14.899484 (MainThread): On master: BEGIN
2021-02-17 23:40:14.899484 (MainThread): Opening a new connection, currently in state closed
2021-02-17 23:40:16.647183 (MainThread): SQL status: SUCCESS 1 in 1.75 seconds
2021-02-17 23:40:16.648178 (MainThread): On master: COMMIT
2021-02-17 23:40:16.648178 (MainThread): Using snowflake connection "master".
2021-02-17 23:40:16.648178 (MainThread): On master: COMMIT
2021-02-17 23:40:18.544161 (MainThread): SQL status: SUCCESS 1 in 1.90 seconds
2021-02-17 23:40:18.544161 (MainThread): 17:40:18 | 
2021-02-17 23:40:18.544161 (MainThread): 17:40:18 | Running 2 on-run-end hooks
2021-02-17 23:40:18.546154 (MainThread): Compiling operation.dbtpoc.dbtpoc-on-run-end-0
2021-02-17 23:40:18.575080 (MainThread): Writing injected SQL for node "operation.dbtpoc.dbtpoc-on-run-end-0"
2021-02-17 23:40:18.585050 (MainThread): 17:40:18 | 1 of 2 START hook: dbtpoc.on-run-end.0............................... [RUN]
2021-02-17 23:40:18.586053 (MainThread): 17:40:18 | 1 of 2 OK hook: dbtpoc.on-run-end.0.................................. [OK in 0.00s]
2021-02-17 23:40:18.588042 (MainThread): Compiling operation.dbtpoc.dbtpoc-on-run-end-1
2021-02-17 23:40:18.607989 (MainThread): Writing injected SQL for node "operation.dbtpoc.dbtpoc-on-run-end-1"
2021-02-17 23:40:18.608987 (MainThread): 17:40:18 | 2 of 2 START hook: dbtpoc.on-run-end.1............................... [RUN]
2021-02-17 23:40:18.609984 (MainThread): 17:40:18 | 2 of 2 OK hook: dbtpoc.on-run-end.1.................................. [OK in 0.00s]
2021-02-17 23:40:18.611979 (MainThread): 17:40:18 | 
2021-02-17 23:40:18.615980 (MainThread): On master: Close
2021-02-17 23:40:19.717397 (MainThread): 17:40:19 | 
2021-02-17 23:40:19.717397 (MainThread): 17:40:19 | Finished running 1 view model, 2 hooks in 18.72s.
2021-02-17 23:40:19.719396 (MainThread): Connection 'master' was properly closed.
2021-02-17 23:40:19.719396 (MainThread): Connection 'list_DBT_POC' was properly closed.
2021-02-17 23:40:19.719396 (MainThread): Connection 'list_DBT_POC_dbt_jkolpak' was properly closed.
2021-02-17 23:40:19.719396 (MainThread): Connection 'model.dbtpoc.model' was properly closed.
2021-02-17 23:40:19.725376 (MainThread): 
2021-02-17 23:40:19.725376 (MainThread): Completed with 1 error and 0 warnings:
2021-02-17 23:40:19.726374 (MainThread): 
2021-02-17 23:40:19.728402 (MainThread): Database Error in model model (models\model.sql)
2021-02-17 23:40:19.729366 (MainThread):   002003 (42S02): SQL compilation error:
2021-02-17 23:40:19.732360 (MainThread):   Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
2021-02-17 23:40:19.733355 (MainThread):   compiled SQL at target\run\dbtpoc\models\model.sql
2021-02-17 23:40:19.734359 (MainThread): 
Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
2021-02-17 23:40:19.735351 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001381B9EEF88>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001381B81DB88>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001381BBE78C8>]}
2021-02-17 23:40:19.735351 (MainThread): Flushing usage events
2021-02-18 00:02:36.777839 (MainThread): Running with dbt=0.18.0
2021-02-18 00:02:37.959068 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, defer=None, exclude=None, fail_fast=False, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='C:\\Users\\jkolpak\\.dbt', project_dir=None, record_timing_info=None, rpc_method='run', selector_name=None, single_threaded=False, state=None, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, use_colors=None, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2021-02-18 00:02:37.960065 (MainThread): Tracking: tracking
2021-02-18 00:02:37.961063 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000145977A4AC8>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000145977B1B48>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000145977B1C88>]}
2021-02-18 00:02:37.987990 (MainThread): Partial parsing not enabled
2021-02-18 00:02:37.988988 (MainThread): Parsing macros\get_custom_schema.sql
2021-02-18 00:02:37.989985 (MainThread): Parsing macros\grant_select_on_schemas.sql
2021-02-18 00:02:37.993975 (MainThread): Parsing macros\adapters.sql
2021-02-18 00:02:38.032054 (MainThread): Parsing macros\catalog.sql
2021-02-18 00:02:38.035028 (MainThread): Parsing macros\materializations\incremental.sql
2021-02-18 00:02:38.047994 (MainThread): Parsing macros\materializations\merge.sql
2021-02-18 00:02:38.051984 (MainThread): Parsing macros\materializations\table.sql
2021-02-18 00:02:38.057967 (MainThread): Parsing macros\materializations\view.sql
2021-02-18 00:02:38.068939 (MainThread): Parsing macros\core.sql
2021-02-18 00:02:38.074926 (MainThread): Parsing macros\adapters\common.sql
2021-02-18 00:02:38.137140 (MainThread): Parsing macros\etc\datetime.sql
2021-02-18 00:02:38.148078 (MainThread): Parsing macros\etc\get_custom_alias.sql
2021-02-18 00:02:38.150073 (MainThread): Parsing macros\etc\get_custom_database.sql
2021-02-18 00:02:38.153065 (MainThread): Parsing macros\etc\get_custom_schema.sql
2021-02-18 00:02:38.156057 (MainThread): Parsing macros\etc\is_incremental.sql
2021-02-18 00:02:38.159049 (MainThread): Parsing macros\etc\query.sql
2021-02-18 00:02:38.161044 (MainThread): Parsing macros\materializations\helpers.sql
2021-02-18 00:02:38.173011 (MainThread): Parsing macros\materializations\common\merge.sql
2021-02-18 00:02:38.191964 (MainThread): Parsing macros\materializations\incremental\helpers.sql
2021-02-18 00:02:38.194955 (MainThread): Parsing macros\materializations\incremental\incremental.sql
2021-02-18 00:02:38.203930 (MainThread): Parsing macros\materializations\seed\seed.sql
2021-02-18 00:02:38.236321 (MainThread): Parsing macros\materializations\snapshot\snapshot.sql
2021-02-18 00:02:38.274873 (MainThread): Parsing macros\materializations\snapshot\snapshot_merge.sql
2021-02-18 00:02:38.277864 (MainThread): Parsing macros\materializations\snapshot\strategies.sql
2021-02-18 00:02:38.302500 (MainThread): Parsing macros\materializations\table\table.sql
2021-02-18 00:02:38.312476 (MainThread): Parsing macros\materializations\view\create_or_replace_view.sql
2021-02-18 00:02:38.318459 (MainThread): Parsing macros\materializations\view\view.sql
2021-02-18 00:02:38.330856 (MainThread): Parsing macros\schema_tests\accepted_values.sql
2021-02-18 00:02:38.335842 (MainThread): Parsing macros\schema_tests\not_null.sql
2021-02-18 00:02:38.337837 (MainThread): Parsing macros\schema_tests\relationships.sql
2021-02-18 00:02:38.340829 (MainThread): Parsing macros\schema_tests\unique.sql
2021-02-18 00:02:38.349389 (MainThread): * Deprecation Warning: dbt v0.17.0 introduces a new config format for the
dbt_project.yml file. Support for the existing version 1 format will be removed
in a future release of dbt. The following packages are currently configured with
config version 1:
 - dbtpoc

For upgrading instructions, consult the documentation:
  https://docs.getdbt.com/docs/guides/migration-guide/upgrading-to-0-17-0

2021-02-18 00:02:38.352246 (MainThread): Sending event: {'category': 'dbt', 'action': 'deprecation', 'label': '002f25d6-06c8-434c-87c3-df60fbbcca99', 'property_': 'warn', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014597706088>]}
2021-02-18 00:02:38.353243 (MainThread): Partial parsing not enabled
2021-02-18 00:02:38.406297 (MainThread): Acquiring new snowflake connection "model.dbtpoc.model".
2021-02-18 00:02:38.453173 (MainThread): Acquiring new snowflake connection "operation.dbtpoc.dbtpoc-on-run-end-0".
2021-02-18 00:02:38.471126 (MainThread): Acquiring new snowflake connection "operation.dbtpoc.dbtpoc-on-run-end-1".
2021-02-18 00:02:38.691192 (MainThread): Found 1 model, 0 tests, 0 snapshots, 0 analyses, 145 macros, 2 operations, 0 seed files, 0 sources
2021-02-18 00:02:38.692190 (MainThread): 
2021-02-18 00:02:38.693188 (MainThread): Acquiring new snowflake connection "master".
2021-02-18 00:02:38.695182 (ThreadPoolExecutor-0_0): Acquiring new snowflake connection "list_DBT_POC".
2021-02-18 00:02:38.713488 (ThreadPoolExecutor-0_0): Using snowflake connection "list_DBT_POC".
2021-02-18 00:02:38.714452 (ThreadPoolExecutor-0_0): On list_DBT_POC: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "list_DBT_POC"} */

    show terse schemas in database DBT_POC
    limit 10000
2021-02-18 00:02:38.714452 (ThreadPoolExecutor-0_0): Opening a new connection, currently in state init
2021-02-18 00:02:40.136003 (ThreadPoolExecutor-0_0): SQL status: SUCCESS 5 in 1.42 seconds
2021-02-18 00:02:40.147108 (ThreadPoolExecutor-0_0): On list_DBT_POC: Close
2021-02-18 00:02:40.618208 (ThreadPoolExecutor-1_0): Acquiring new snowflake connection "list_DBT_POC_dbt_jkolpak".
2021-02-18 00:02:40.624609 (ThreadPoolExecutor-1_0): Using snowflake connection "list_DBT_POC_dbt_jkolpak".
2021-02-18 00:02:40.624609 (ThreadPoolExecutor-1_0): On list_DBT_POC_dbt_jkolpak: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "list_DBT_POC_dbt_jkolpak"} */

    show terse objects in DBT_POC.dbt_jkolpak
2021-02-18 00:02:40.624609 (ThreadPoolExecutor-1_0): Opening a new connection, currently in state init
2021-02-18 00:02:42.157894 (ThreadPoolExecutor-1_0): SQL status: SUCCESS 5 in 1.53 seconds
2021-02-18 00:02:42.162881 (ThreadPoolExecutor-1_0): On list_DBT_POC_dbt_jkolpak: Close
2021-02-18 00:02:42.563326 (MainThread): Using snowflake connection "master".
2021-02-18 00:02:42.564323 (MainThread): On master: BEGIN
2021-02-18 00:02:42.564323 (MainThread): Opening a new connection, currently in state init
2021-02-18 00:02:43.658672 (MainThread): SQL status: SUCCESS 1 in 1.09 seconds
2021-02-18 00:02:43.658672 (MainThread): On master: COMMIT
2021-02-18 00:02:43.658672 (MainThread): Using snowflake connection "master".
2021-02-18 00:02:43.658672 (MainThread): On master: COMMIT
2021-02-18 00:02:44.020789 (MainThread): SQL status: SUCCESS 1 in 0.36 seconds
2021-02-18 00:02:44.021787 (MainThread): On master: Close
2021-02-18 00:02:44.298139 (MainThread): 18:02:44 | Concurrency: 16 threads (target='dev')
2021-02-18 00:02:44.299147 (MainThread): 18:02:44 | 
2021-02-18 00:02:44.305124 (Thread-1): Began running node model.dbtpoc.model
2021-02-18 00:02:44.306121 (Thread-1): 18:02:44 | 1 of 1 START view model dbt_jkolpak.model............................ [RUN]
2021-02-18 00:02:44.307118 (Thread-1): Acquiring new snowflake connection "model.dbtpoc.model".
2021-02-18 00:02:44.307118 (Thread-1): Compiling model.dbtpoc.model
2021-02-18 00:02:44.324148 (Thread-1): Writing injected SQL for node "model.dbtpoc.model"
2021-02-18 00:02:44.325145 (Thread-1): finished collecting timing info
2021-02-18 00:02:44.358155 (Thread-1): Writing runtime SQL for node "model.dbtpoc.model"
2021-02-18 00:02:44.360117 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-18 00:02:44.360117 (Thread-1): On model.dbtpoc.model: BEGIN
2021-02-18 00:02:44.360117 (Thread-1): Opening a new connection, currently in state init
2021-02-18 00:02:45.246179 (Thread-1): SQL status: SUCCESS 1 in 0.89 seconds
2021-02-18 00:02:45.246179 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-18 00:02:45.246179 (Thread-1): On model.dbtpoc.model: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "node_id": "model.dbtpoc.model"} */

  create or replace  view DBT_POC.dbt_jkolpak.model  as (
    SELECT * FROM table_taco
  );
2021-02-18 00:02:45.540500 (Thread-1): Snowflake query id: 019a5c82-052a-188a-0000-40150030404e
2021-02-18 00:02:45.540500 (Thread-1): Snowflake error: 002003 (42S02): SQL compilation error:
Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
2021-02-18 00:02:45.540500 (Thread-1): finished collecting timing info
2021-02-18 00:02:45.541498 (Thread-1): On model.dbtpoc.model: ROLLBACK
2021-02-18 00:02:46.013221 (Thread-1): On model.dbtpoc.model: Close
2021-02-18 00:02:46.371107 (Thread-1): Database Error in model model (models\model.sql)
  002003 (42S02): SQL compilation error:
  Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
  compiled SQL at target\run\dbtpoc\models\model.sql
Traceback (most recent call last):
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 161, in exception_handler
    yield
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 78, in add_query
    cursor.execute(sql, bindings)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\cursor.py", line 596, in execute
    errvalue)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\errors.py", line 123, in errorhandler_wrapper
    cursor.errorhandler(connection, cursor, error_class, error_value)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\errors.py", line 88, in default_errorhandler
    done_format_msg=error_value.get('done_format_msg'))
snowflake.connector.errors.ProgrammingError: 002003 (42S02): SQL compilation error:
Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\base.py", line 333, in safe_run
    result = self.compile_and_execute(manifest, ctx)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\base.py", line 276, in compile_and_execute
    result = self.run(ctx.node, manifest)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\base.py", line 378, in run
    return self.execute(compiled_node, manifest)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\run.py", line 233, in execute
    result = MacroGenerator(materialization_macro, context)()
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 27, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 65, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 327, in __call__
    return self.call_macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\clients\jinja.py", line 257, in call_macro
    return macro(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 675, in __call__
    return self._invoke(arguments, autoescape)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "<template>", line 41, in macro
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\sandbox.py", line 462, in call
    return __context.call(__obj, *args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\jinja2\runtime.py", line 290, in call
    return __obj(*args, **kwargs)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\base\impl.py", line 227, in execute
    fetch=fetch
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 123, in execute
    _, cursor = self.add_query(sql, auto_begin)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 312, in add_query
    abridge_sql_log=abridge_sql_log
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 86, in add_query
    return connection, cursor
  File "C:\Users\jkolpak\AppData\Local\Continuum\anaconda3\lib\contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 178, in exception_handler
    raise DatabaseException(msg)
dbt.exceptions.DatabaseException: Database Error in model model (models\model.sql)
  002003 (42S02): SQL compilation error:
  Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
  compiled SQL at target\run\dbtpoc\models\model.sql
2021-02-18 00:02:46.381079 (Thread-1): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': '002f25d6-06c8-434c-87c3-df60fbbcca99', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014597BB3948>]}
2021-02-18 00:02:46.383075 (Thread-1): 18:02:46 | 1 of 1 ERROR creating view model dbt_jkolpak.model................... [ERROR in 2.07s]
2021-02-18 00:02:46.384072 (Thread-1): Finished running node model.dbtpoc.model
2021-02-18 00:02:46.428956 (MainThread): Acquiring new snowflake connection "master".
2021-02-18 00:02:46.430372 (MainThread): Using snowflake connection "master".
2021-02-18 00:02:46.430372 (MainThread): On master: BEGIN
2021-02-18 00:02:46.430372 (MainThread): Opening a new connection, currently in state closed
2021-02-18 00:02:47.385928 (MainThread): SQL status: SUCCESS 1 in 0.96 seconds
2021-02-18 00:02:47.385928 (MainThread): On master: COMMIT
2021-02-18 00:02:47.385928 (MainThread): Using snowflake connection "master".
2021-02-18 00:02:47.386925 (MainThread): On master: COMMIT
2021-02-18 00:02:47.982988 (MainThread): SQL status: SUCCESS 1 in 0.60 seconds
2021-02-18 00:02:47.982988 (MainThread): 18:02:47 | 
2021-02-18 00:02:47.983962 (MainThread): 18:02:47 | Running 2 on-run-end hooks
2021-02-18 00:02:47.983962 (MainThread): Compiling operation.dbtpoc.dbtpoc-on-run-end-0
2021-02-18 00:02:47.996926 (MainThread): Writing injected SQL for node "operation.dbtpoc.dbtpoc-on-run-end-0"
2021-02-18 00:02:47.999845 (MainThread): 18:02:48 | 1 of 2 START hook: dbtpoc.on-run-end.0............................... [RUN]
2021-02-18 00:02:48.000913 (MainThread): 18:02:48 | 1 of 2 OK hook: dbtpoc.on-run-end.0.................................. [OK in 0.00s]
2021-02-18 00:02:48.002838 (MainThread): Compiling operation.dbtpoc.dbtpoc-on-run-end-1
2021-02-18 00:02:48.007824 (MainThread): Writing injected SQL for node "operation.dbtpoc.dbtpoc-on-run-end-1"
2021-02-18 00:02:48.008821 (MainThread): 18:02:48 | 2 of 2 START hook: dbtpoc.on-run-end.1............................... [RUN]
2021-02-18 00:02:48.009834 (MainThread): 18:02:48 | 2 of 2 OK hook: dbtpoc.on-run-end.1.................................. [OK in 0.00s]
2021-02-18 00:02:48.015808 (MainThread): 18:02:48 | 
2021-02-18 00:02:48.016807 (MainThread): On master: Close
2021-02-18 00:02:48.316495 (MainThread): 18:02:48 | 
2021-02-18 00:02:48.317493 (MainThread): 18:02:48 | Finished running 1 view model, 2 hooks in 9.62s.
2021-02-18 00:02:48.317493 (MainThread): Connection 'master' was properly closed.
2021-02-18 00:02:48.317493 (MainThread): Connection 'list_DBT_POC' was properly closed.
2021-02-18 00:02:48.317493 (MainThread): Connection 'list_DBT_POC_dbt_jkolpak' was properly closed.
2021-02-18 00:02:48.317493 (MainThread): Connection 'model.dbtpoc.model' was properly closed.
2021-02-18 00:02:48.322395 (MainThread): 
2021-02-18 00:02:48.324390 (MainThread): Completed with 1 error and 0 warnings:
2021-02-18 00:02:48.324390 (MainThread): 
2021-02-18 00:02:48.325387 (MainThread): Database Error in model model (models\model.sql)
2021-02-18 00:02:48.326386 (MainThread):   002003 (42S02): SQL compilation error:
2021-02-18 00:02:48.326386 (MainThread):   Object 'DBT_POC.DBT_JKOLPAK.TABLE_TACO' does not exist or not authorized.
2021-02-18 00:02:48.331370 (MainThread):   compiled SQL at target\run\dbtpoc\models\model.sql
2021-02-18 00:02:48.332370 (MainThread): 
Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
2021-02-18 00:02:48.333368 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001459791C9C8>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014597A1AB48>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000014597A00F48>]}
2021-02-18 00:02:48.333368 (MainThread): Flushing usage events
2021-02-18 00:03:55.647912 (MainThread): Running with dbt=0.18.0
2021-02-18 00:03:56.491396 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, defer=None, exclude=None, fail_fast=False, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='C:\\Users\\jkolpak\\.dbt', project_dir=None, record_timing_info=None, rpc_method='run', selector_name=None, single_threaded=False, state=None, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, use_colors=None, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2021-02-18 00:03:56.493370 (MainThread): Tracking: tracking
2021-02-18 00:03:56.493370 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EFA09B1048>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EFA4A34A08>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EFA4A34848>]}
2021-02-18 00:03:56.540254 (MainThread): Partial parsing not enabled
2021-02-18 00:03:56.542248 (MainThread): Parsing macros\get_custom_schema.sql
2021-02-18 00:03:56.544241 (MainThread): Parsing macros\grant_select_on_schemas.sql
2021-02-18 00:03:56.549228 (MainThread): Parsing macros\adapters.sql
2021-02-18 00:03:56.598790 (MainThread): Parsing macros\catalog.sql
2021-02-18 00:03:56.601782 (MainThread): Parsing macros\materializations\incremental.sql
2021-02-18 00:03:56.622726 (MainThread): Parsing macros\materializations\merge.sql
2021-02-18 00:03:56.626715 (MainThread): Parsing macros\materializations\table.sql
2021-02-18 00:03:56.632699 (MainThread): Parsing macros\materializations\view.sql
2021-02-18 00:03:56.642281 (MainThread): Parsing macros\core.sql
2021-02-18 00:03:56.652229 (MainThread): Parsing macros\adapters\common.sql
2021-02-18 00:03:56.761442 (MainThread): Parsing macros\etc\datetime.sql
2021-02-18 00:03:56.784620 (MainThread): Parsing macros\etc\get_custom_alias.sql
2021-02-18 00:03:56.788611 (MainThread): Parsing macros\etc\get_custom_database.sql
2021-02-18 00:03:56.794594 (MainThread): Parsing macros\etc\get_custom_schema.sql
2021-02-18 00:03:56.798583 (MainThread): Parsing macros\etc\is_incremental.sql
2021-02-18 00:03:56.803570 (MainThread): Parsing macros\etc\query.sql
2021-02-18 00:03:56.806561 (MainThread): Parsing macros\materializations\helpers.sql
2021-02-18 00:03:56.828909 (MainThread): Parsing macros\materializations\common\merge.sql
2021-02-18 00:03:56.857185 (MainThread): Parsing macros\materializations\incremental\helpers.sql
2021-02-18 00:03:56.862175 (MainThread): Parsing macros\materializations\incremental\incremental.sql
2021-02-18 00:03:56.873146 (MainThread): Parsing macros\materializations\seed\seed.sql
2021-02-18 00:03:56.913544 (MainThread): Parsing macros\materializations\snapshot\snapshot.sql
2021-02-18 00:03:56.968671 (MainThread): Parsing macros\materializations\snapshot\snapshot_merge.sql
2021-02-18 00:03:56.973288 (MainThread): Parsing macros\materializations\snapshot\strategies.sql
2021-02-18 00:03:57.011362 (MainThread): Parsing macros\materializations\table\table.sql
2021-02-18 00:03:57.023331 (MainThread): Parsing macros\materializations\view\create_or_replace_view.sql
2021-02-18 00:03:57.032307 (MainThread): Parsing macros\materializations\view\view.sql
2021-02-18 00:03:57.043278 (MainThread): Parsing macros\schema_tests\accepted_values.sql
2021-02-18 00:03:57.048264 (MainThread): Parsing macros\schema_tests\not_null.sql
2021-02-18 00:03:57.050259 (MainThread): Parsing macros\schema_tests\relationships.sql
2021-02-18 00:03:57.054250 (MainThread): Parsing macros\schema_tests\unique.sql
2021-02-18 00:03:57.065892 (MainThread): * Deprecation Warning: dbt v0.17.0 introduces a new config format for the
dbt_project.yml file. Support for the existing version 1 format will be removed
in a future release of dbt. The following packages are currently configured with
config version 1:
 - dbtpoc

For upgrading instructions, consult the documentation:
  https://docs.getdbt.com/docs/guides/migration-guide/upgrading-to-0-17-0

2021-02-18 00:03:57.075865 (MainThread): Sending event: {'category': 'dbt', 'action': 'deprecation', 'label': 'cf2826d3-a770-4396-8b9d-d647f6b0f99b', 'property_': 'warn', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EFA4994D88>]}
2021-02-18 00:03:57.077861 (MainThread): Partial parsing not enabled
2021-02-18 00:03:57.148886 (MainThread): Acquiring new snowflake connection "model.dbtpoc.model".
2021-02-18 00:03:57.183466 (MainThread): Acquiring new snowflake connection "operation.dbtpoc.dbtpoc-on-run-end-0".
2021-02-18 00:03:57.194901 (MainThread): Acquiring new snowflake connection "operation.dbtpoc.dbtpoc-on-run-end-1".
2021-02-18 00:03:57.441995 (MainThread): Found 1 model, 0 tests, 0 snapshots, 0 analyses, 145 macros, 2 operations, 0 seed files, 0 sources
2021-02-18 00:03:57.442991 (MainThread): 
2021-02-18 00:03:57.444986 (MainThread): Acquiring new snowflake connection "master".
2021-02-18 00:03:57.447977 (ThreadPoolExecutor-0_0): Acquiring new snowflake connection "list_DBT_POC".
2021-02-18 00:03:57.486874 (ThreadPoolExecutor-0_0): Using snowflake connection "list_DBT_POC".
2021-02-18 00:03:57.487872 (ThreadPoolExecutor-0_0): On list_DBT_POC: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "list_DBT_POC"} */

    show terse schemas in database DBT_POC
    limit 10000
2021-02-18 00:03:57.487872 (ThreadPoolExecutor-0_0): Opening a new connection, currently in state init
2021-02-18 00:03:58.721689 (ThreadPoolExecutor-0_0): SQL status: SUCCESS 5 in 1.23 seconds
2021-02-18 00:03:58.725707 (ThreadPoolExecutor-0_0): On list_DBT_POC: Close
2021-02-18 00:03:59.076890 (ThreadPoolExecutor-1_0): Acquiring new snowflake connection "list_DBT_POC_dbt_jkolpak".
2021-02-18 00:03:59.084356 (ThreadPoolExecutor-1_0): Using snowflake connection "list_DBT_POC_dbt_jkolpak".
2021-02-18 00:03:59.084356 (ThreadPoolExecutor-1_0): On list_DBT_POC_dbt_jkolpak: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "list_DBT_POC_dbt_jkolpak"} */

    show terse objects in DBT_POC.dbt_jkolpak
2021-02-18 00:03:59.084356 (ThreadPoolExecutor-1_0): Opening a new connection, currently in state init
2021-02-18 00:04:00.188967 (ThreadPoolExecutor-1_0): SQL status: SUCCESS 5 in 1.10 seconds
2021-02-18 00:04:00.194013 (ThreadPoolExecutor-1_0): On list_DBT_POC_dbt_jkolpak: Close
2021-02-18 00:04:00.548861 (MainThread): Using snowflake connection "master".
2021-02-18 00:04:00.548861 (MainThread): On master: BEGIN
2021-02-18 00:04:00.548861 (MainThread): Opening a new connection, currently in state init
2021-02-18 00:04:01.492313 (MainThread): SQL status: SUCCESS 1 in 0.94 seconds
2021-02-18 00:04:01.492313 (MainThread): On master: COMMIT
2021-02-18 00:04:01.492313 (MainThread): Using snowflake connection "master".
2021-02-18 00:04:01.492313 (MainThread): On master: COMMIT
2021-02-18 00:04:02.366263 (MainThread): SQL status: SUCCESS 1 in 0.87 seconds
2021-02-18 00:04:02.366263 (MainThread): On master: Close
2021-02-18 00:04:02.651367 (MainThread): 18:04:02 | Concurrency: 16 threads (target='dev')
2021-02-18 00:04:02.651367 (MainThread): 18:04:02 | 
2021-02-18 00:04:02.657345 (Thread-1): Began running node model.dbtpoc.model
2021-02-18 00:04:02.659340 (Thread-1): 18:04:02 | 1 of 1 START view model dbt_jkolpak.model............................ [RUN]
2021-02-18 00:04:02.660373 (Thread-1): Acquiring new snowflake connection "model.dbtpoc.model".
2021-02-18 00:04:02.660373 (Thread-1): Compiling model.dbtpoc.model
2021-02-18 00:04:02.693998 (Thread-1): Writing injected SQL for node "model.dbtpoc.model"
2021-02-18 00:04:02.694982 (Thread-1): finished collecting timing info
2021-02-18 00:04:02.732369 (Thread-1): Writing runtime SQL for node "model.dbtpoc.model"
2021-02-18 00:04:02.734396 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-18 00:04:02.734396 (Thread-1): On model.dbtpoc.model: BEGIN
2021-02-18 00:04:02.734396 (Thread-1): Opening a new connection, currently in state init
2021-02-18 00:04:03.725202 (Thread-1): SQL status: SUCCESS 1 in 0.99 seconds
2021-02-18 00:04:03.726230 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-18 00:04:03.726230 (Thread-1): On model.dbtpoc.model: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "node_id": "model.dbtpoc.model"} */

  create or replace  view DBT_POC.dbt_jkolpak.model  as (
    SELECT S_SUPPKEY AS SUPPLIER_KEY
    , S_NAME AS SUPPLIER_NAME
    , S_ADDRESS AS SUPPLIER_ADDRESS
    , S_PHONE AS SUPPLIER_PHONE_NUMBER
    , S_ACCTBAL AS SUPPLIER_ACCOUNT_BALANCE
    , S_COMMENT AS SUPPLIER_COMMENT
    , N.N_NAME AS SUPPLIER_NATION
    , R.R_NAME AS SUPPLIER_REGION
FROM "SNOWFLAKE_SAMPLE_DATA"."TPCH_SF10"."SUPPLIER" S
LEFT JOIN "SNOWFLAKE_SAMPLE_DATA"."TPCH_SF10"."NATION" N ON S.S_NATIONKEY = N.N_NATIONKEY
LEFT JOIN "SNOWFLAKE_SAMPLE_DATA"."TPCH_SF10"."REGION" R ON N.N_REGIONKEY = R.R_REGIONKEY
  );
2021-02-18 00:04:05.840400 (Thread-1): SQL status: SUCCESS 1 in 2.11 seconds
2021-02-18 00:04:05.841278 (Thread-1): On model.dbtpoc.model: COMMIT
2021-02-18 00:04:05.841278 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-18 00:04:05.841278 (Thread-1): On model.dbtpoc.model: COMMIT
2021-02-18 00:04:06.038100 (Thread-1): SQL status: SUCCESS 1 in 0.20 seconds
2021-02-18 00:04:06.048073 (Thread-1): finished collecting timing info
2021-02-18 00:04:06.049071 (Thread-1): On model.dbtpoc.model: Close
2021-02-18 00:04:06.674434 (Thread-1): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': 'cf2826d3-a770-4396-8b9d-d647f6b0f99b', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EFA4E40088>]}
2021-02-18 00:04:06.675431 (Thread-1): 18:04:06 | 1 of 1 OK created view model dbt_jkolpak.model....................... [SUCCESS 1 in 4.01s]
2021-02-18 00:04:06.676395 (Thread-1): Finished running node model.dbtpoc.model
2021-02-18 00:04:06.698127 (MainThread): Acquiring new snowflake connection "master".
2021-02-18 00:04:06.698127 (MainThread): Using snowflake connection "master".
2021-02-18 00:04:06.698127 (MainThread): On master: BEGIN
2021-02-18 00:04:06.698127 (MainThread): Opening a new connection, currently in state closed
2021-02-18 00:04:07.644873 (MainThread): SQL status: SUCCESS 1 in 0.95 seconds
2021-02-18 00:04:07.644873 (MainThread): On master: COMMIT
2021-02-18 00:04:07.645871 (MainThread): Using snowflake connection "master".
2021-02-18 00:04:07.645871 (MainThread): On master: COMMIT
2021-02-18 00:04:07.945460 (MainThread): SQL status: SUCCESS 1 in 0.30 seconds
2021-02-18 00:04:07.945460 (MainThread): 18:04:07 | 
2021-02-18 00:04:07.946458 (MainThread): 18:04:07 | Running 2 on-run-end hooks
2021-02-18 00:04:07.946458 (MainThread): Compiling operation.dbtpoc.dbtpoc-on-run-end-0
2021-02-18 00:04:07.967405 (MainThread): Writing injected SQL for node "operation.dbtpoc.dbtpoc-on-run-end-0"
2021-02-18 00:04:07.975389 (MainThread): 18:04:07 | 1 of 2 START hook: dbtpoc.on-run-end.0............................... [RUN]
2021-02-18 00:04:07.976383 (MainThread): Using snowflake connection "master".
2021-02-18 00:04:07.976383 (MainThread): On master: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "master"} */

  
    grant usage on schema dbt_jkolpak to role reporter;
2021-02-18 00:04:08.171952 (MainThread): Snowflake query id: 019a5c84-0543-22b2-0000-40150030209a
2021-02-18 00:04:08.171952 (MainThread): Snowflake error: 002003 (02000): SQL compilation error:
Role 'REPORTER' does not exist or not authorized.
2021-02-18 00:04:08.171952 (MainThread): Database error while running on-run-end
2021-02-18 00:04:08.172948 (MainThread): On master: Close
2021-02-18 00:04:08.658169 (MainThread): Connection 'master' was properly closed.
2021-02-18 00:04:08.658169 (MainThread): Connection 'list_DBT_POC' was properly closed.
2021-02-18 00:04:08.658169 (MainThread): Connection 'list_DBT_POC_dbt_jkolpak' was properly closed.
2021-02-18 00:04:08.658169 (MainThread): Connection 'model.dbtpoc.model' was properly closed.
2021-02-18 00:04:08.659166 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EFA4D5B208>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EFA4D51188>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EFA4E43548>]}
2021-02-18 00:04:08.659166 (MainThread): Flushing usage events
2021-02-18 00:04:09.050571 (MainThread): Encountered an error:
2021-02-18 00:04:09.057555 (MainThread): Database Error
  002003 (02000): SQL compilation error:
  Role 'REPORTER' does not exist or not authorized.
2021-02-18 00:04:09.077698 (MainThread): Traceback (most recent call last):
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 161, in exception_handler
    yield
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 78, in add_query
    cursor.execute(sql, bindings)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\cursor.py", line 596, in execute
    errvalue)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\errors.py", line 123, in errorhandler_wrapper
    cursor.errorhandler(connection, cursor, error_class, error_value)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\snowflake\connector\errors.py", line 88, in default_errorhandler
    done_format_msg=error_value.get('done_format_msg'))
snowflake.connector.errors.ProgrammingError: 002003 (02000): SQL compilation error:
Role 'REPORTER' does not exist or not authorized.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\main.py", line 124, in main
    results, succeeded = handle_and_check(args)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\main.py", line 202, in handle_and_check
    task, res = run_from_args(parsed)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\main.py", line 255, in run_from_args
    results = task.run()
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\runnable.py", line 419, in run
    result = self.execute_with_hooks(selected_uids)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\runnable.py", line 381, in execute_with_hooks
    self.after_run(adapter, res)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\run.py", line 415, in after_run
    self.safe_run_hooks(adapter, RunHookType.End, extras)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\run.py", line 337, in safe_run_hooks
    self.run_hooks(adapter, hook_type, extra_context)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\task\run.py", line 320, in run_hooks
    fetch=False)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\base\impl.py", line 227, in execute
    fetch=fetch
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 123, in execute
    _, cursor = self.add_query(sql, auto_begin)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 312, in add_query
    abridge_sql_log=abridge_sql_log
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\sql\connections.py", line 86, in add_query
    return connection, cursor
  File "C:\Users\jkolpak\AppData\Local\Continuum\anaconda3\lib\contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "c:\users\jkolpak\dbt-venv\lib\site-packages\dbt\adapters\snowflake\connections.py", line 178, in exception_handler
    raise DatabaseException(msg)
dbt.exceptions.DatabaseException: Database Error
  002003 (02000): SQL compilation error:
  Role 'REPORTER' does not exist or not authorized.

2021-02-18 00:05:02.717336 (MainThread): Running with dbt=0.18.0
2021-02-18 00:05:03.609827 (MainThread): running dbt with arguments Namespace(cls=<class 'dbt.task.run.RunTask'>, debug=False, defer=None, exclude=None, fail_fast=False, full_refresh=False, log_cache_events=False, log_format='default', models=None, partial_parse=None, profile=None, profiles_dir='C:\\Users\\jkolpak\\.dbt', project_dir=None, record_timing_info=None, rpc_method='run', selector_name=None, single_threaded=False, state=None, strict=False, target=None, test_new_parser=False, threads=None, use_cache=True, use_colors=None, vars='{}', version_check=True, warn_error=False, which='run', write_json=True)
2021-02-18 00:05:03.610824 (MainThread): Tracking: tracking
2021-02-18 00:05:03.610824 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017FEC5B1B88>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017FE8470C08>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017FEC5BF248>]}
2021-02-18 00:05:03.645736 (MainThread): Partial parsing not enabled
2021-02-18 00:05:03.646734 (MainThread): Parsing macros\get_custom_schema.sql
2021-02-18 00:05:03.647731 (MainThread): Parsing macros\grant_select_on_schemas.sql
2021-02-18 00:05:03.650723 (MainThread): Parsing macros\adapters.sql
2021-02-18 00:05:03.696724 (MainThread): Parsing macros\catalog.sql
2021-02-18 00:05:03.698718 (MainThread): Parsing macros\materializations\incremental.sql
2021-02-18 00:05:03.711684 (MainThread): Parsing macros\materializations\merge.sql
2021-02-18 00:05:03.714676 (MainThread): Parsing macros\materializations\table.sql
2021-02-18 00:05:03.720659 (MainThread): Parsing macros\materializations\view.sql
2021-02-18 00:05:03.725646 (MainThread): Parsing macros\core.sql
2021-02-18 00:05:03.730633 (MainThread): Parsing macros\adapters\common.sql
2021-02-18 00:05:03.791659 (MainThread): Parsing macros\etc\datetime.sql
2021-02-18 00:05:03.805996 (MainThread): Parsing macros\etc\get_custom_alias.sql
2021-02-18 00:05:03.807991 (MainThread): Parsing macros\etc\get_custom_database.sql
2021-02-18 00:05:03.810983 (MainThread): Parsing macros\etc\get_custom_schema.sql
2021-02-18 00:05:03.814972 (MainThread): Parsing macros\etc\is_incremental.sql
2021-02-18 00:05:03.818962 (MainThread): Parsing macros\etc\query.sql
2021-02-18 00:05:03.820957 (MainThread): Parsing macros\materializations\helpers.sql
2021-02-18 00:05:03.837822 (MainThread): Parsing macros\materializations\common\merge.sql
2021-02-18 00:05:03.873725 (MainThread): Parsing macros\materializations\incremental\helpers.sql
2021-02-18 00:05:03.876717 (MainThread): Parsing macros\materializations\incremental\incremental.sql
2021-02-18 00:05:03.889682 (MainThread): Parsing macros\materializations\seed\seed.sql
2021-02-18 00:05:03.930499 (MainThread): Parsing macros\materializations\snapshot\snapshot.sql
2021-02-18 00:05:03.982891 (MainThread): Parsing macros\materializations\snapshot\snapshot_merge.sql
2021-02-18 00:05:03.985880 (MainThread): Parsing macros\materializations\snapshot\strategies.sql
2021-02-18 00:05:04.008753 (MainThread): Parsing macros\materializations\table\table.sql
2021-02-18 00:05:04.018541 (MainThread): Parsing macros\materializations\view\create_or_replace_view.sql
2021-02-18 00:05:04.025099 (MainThread): Parsing macros\materializations\view\view.sql
2021-02-18 00:05:04.035736 (MainThread): Parsing macros\schema_tests\accepted_values.sql
2021-02-18 00:05:04.039403 (MainThread): Parsing macros\schema_tests\not_null.sql
2021-02-18 00:05:04.041896 (MainThread): Parsing macros\schema_tests\relationships.sql
2021-02-18 00:05:04.043919 (MainThread): Parsing macros\schema_tests\unique.sql
2021-02-18 00:05:04.052869 (MainThread): * Deprecation Warning: dbt v0.17.0 introduces a new config format for the
dbt_project.yml file. Support for the existing version 1 format will be removed
in a future release of dbt. The following packages are currently configured with
config version 1:
 - dbtpoc

For upgrading instructions, consult the documentation:
  https://docs.getdbt.com/docs/guides/migration-guide/upgrading-to-0-17-0

2021-02-18 00:05:04.060812 (MainThread): Sending event: {'category': 'dbt', 'action': 'deprecation', 'label': '1ec98fb6-4c2b-4845-a02b-6b5e93b8b75c', 'property_': 'warn', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017FEC60B788>]}
2021-02-18 00:05:04.061810 (MainThread): Partial parsing not enabled
2021-02-18 00:05:04.123089 (MainThread): Acquiring new snowflake connection "model.dbtpoc.model".
2021-02-18 00:05:04.358950 (MainThread): Found 1 model, 0 tests, 0 snapshots, 0 analyses, 145 macros, 0 operations, 0 seed files, 0 sources
2021-02-18 00:05:04.359948 (MainThread): 
2021-02-18 00:05:04.360945 (MainThread): Acquiring new snowflake connection "master".
2021-02-18 00:05:04.364934 (ThreadPoolExecutor-0_0): Acquiring new snowflake connection "list_DBT_POC".
2021-02-18 00:05:04.445543 (ThreadPoolExecutor-0_0): Using snowflake connection "list_DBT_POC".
2021-02-18 00:05:04.445543 (ThreadPoolExecutor-0_0): On list_DBT_POC: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "list_DBT_POC"} */

    show terse schemas in database DBT_POC
    limit 10000
2021-02-18 00:05:04.445543 (ThreadPoolExecutor-0_0): Opening a new connection, currently in state init
2021-02-18 00:05:06.116758 (ThreadPoolExecutor-0_0): SQL status: SUCCESS 5 in 1.67 seconds
2021-02-18 00:05:06.120779 (ThreadPoolExecutor-0_0): On list_DBT_POC: Close
2021-02-18 00:05:06.416058 (ThreadPoolExecutor-1_0): Acquiring new snowflake connection "list_DBT_POC_dbt_jkolpak".
2021-02-18 00:05:06.428027 (ThreadPoolExecutor-1_0): Using snowflake connection "list_DBT_POC_dbt_jkolpak".
2021-02-18 00:05:06.428027 (ThreadPoolExecutor-1_0): On list_DBT_POC_dbt_jkolpak: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "connection_name": "list_DBT_POC_dbt_jkolpak"} */

    show terse objects in DBT_POC.dbt_jkolpak
2021-02-18 00:05:06.428027 (ThreadPoolExecutor-1_0): Opening a new connection, currently in state init
2021-02-18 00:05:07.734635 (ThreadPoolExecutor-1_0): SQL status: SUCCESS 6 in 1.31 seconds
2021-02-18 00:05:07.741267 (ThreadPoolExecutor-1_0): On list_DBT_POC_dbt_jkolpak: Close
2021-02-18 00:05:08.134019 (MainThread): Using snowflake connection "master".
2021-02-18 00:05:08.134019 (MainThread): On master: BEGIN
2021-02-18 00:05:08.135015 (MainThread): Opening a new connection, currently in state init
2021-02-18 00:05:09.104840 (MainThread): SQL status: SUCCESS 1 in 0.97 seconds
2021-02-18 00:05:09.104840 (MainThread): On master: COMMIT
2021-02-18 00:05:09.105837 (MainThread): Using snowflake connection "master".
2021-02-18 00:05:09.105837 (MainThread): On master: COMMIT
2021-02-18 00:05:09.617874 (MainThread): SQL status: SUCCESS 1 in 0.51 seconds
2021-02-18 00:05:09.617874 (MainThread): On master: Close
2021-02-18 00:05:10.117757 (MainThread): 18:05:10 | Concurrency: 16 threads (target='dev')
2021-02-18 00:05:10.118753 (MainThread): 18:05:10 | 
2021-02-18 00:05:10.122550 (Thread-1): Began running node model.dbtpoc.model
2021-02-18 00:05:10.123548 (Thread-1): 18:05:10 | 1 of 1 START view model dbt_jkolpak.model............................ [RUN]
2021-02-18 00:05:10.128534 (Thread-1): Acquiring new snowflake connection "model.dbtpoc.model".
2021-02-18 00:05:10.129533 (Thread-1): Compiling model.dbtpoc.model
2021-02-18 00:05:10.150220 (Thread-1): Writing injected SQL for node "model.dbtpoc.model"
2021-02-18 00:05:10.151224 (Thread-1): finished collecting timing info
2021-02-18 00:05:10.194913 (Thread-1): Writing runtime SQL for node "model.dbtpoc.model"
2021-02-18 00:05:10.197906 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-18 00:05:10.198903 (Thread-1): On model.dbtpoc.model: BEGIN
2021-02-18 00:05:10.198903 (Thread-1): Opening a new connection, currently in state init
2021-02-18 00:05:11.132062 (Thread-1): SQL status: SUCCESS 1 in 0.93 seconds
2021-02-18 00:05:11.132062 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-18 00:05:11.132062 (Thread-1): On model.dbtpoc.model: /* {"app": "dbt", "dbt_version": "0.18.0", "profile_name": "dbtpoc", "target_name": "dev", "node_id": "model.dbtpoc.model"} */

  create or replace  view DBT_POC.dbt_jkolpak.model  as (
    SELECT S_SUPPKEY AS SUPPLIER_KEY
    , S_NAME AS SUPPLIER_NAME
    , S_ADDRESS AS SUPPLIER_ADDRESS
    , S_PHONE AS SUPPLIER_PHONE_NUMBER
    , S_ACCTBAL AS SUPPLIER_ACCOUNT_BALANCE
    , S_COMMENT AS SUPPLIER_COMMENT
    , N.N_NAME AS SUPPLIER_NATION
    , R.R_NAME AS SUPPLIER_REGION
FROM "SNOWFLAKE_SAMPLE_DATA"."TPCH_SF10"."SUPPLIER" S
LEFT JOIN "SNOWFLAKE_SAMPLE_DATA"."TPCH_SF10"."NATION" N ON S.S_NATIONKEY = N.N_NATIONKEY
LEFT JOIN "SNOWFLAKE_SAMPLE_DATA"."TPCH_SF10"."REGION" R ON N.N_REGIONKEY = R.R_REGIONKEY
  );
2021-02-18 00:05:12.180151 (Thread-1): SQL status: SUCCESS 1 in 1.05 seconds
2021-02-18 00:05:12.181118 (Thread-1): On model.dbtpoc.model: COMMIT
2021-02-18 00:05:12.181118 (Thread-1): Using snowflake connection "model.dbtpoc.model".
2021-02-18 00:05:12.181118 (Thread-1): On model.dbtpoc.model: COMMIT
2021-02-18 00:05:12.361572 (Thread-1): SQL status: SUCCESS 1 in 0.18 seconds
2021-02-18 00:05:12.373418 (Thread-1): finished collecting timing info
2021-02-18 00:05:12.373418 (Thread-1): On model.dbtpoc.model: Close
2021-02-18 00:05:12.722293 (Thread-1): Sending event: {'category': 'dbt', 'action': 'run_model', 'label': '1ec98fb6-4c2b-4845-a02b-6b5e93b8b75c', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017FEC7D1B08>]}
2021-02-18 00:05:12.724287 (Thread-1): 18:05:12 | 1 of 1 OK created view model dbt_jkolpak.model....................... [SUCCESS 1 in 2.59s]
2021-02-18 00:05:12.725285 (Thread-1): Finished running node model.dbtpoc.model
2021-02-18 00:05:12.750913 (MainThread): Acquiring new snowflake connection "master".
2021-02-18 00:05:12.750913 (MainThread): Using snowflake connection "master".
2021-02-18 00:05:12.750913 (MainThread): On master: BEGIN
2021-02-18 00:05:12.750913 (MainThread): Opening a new connection, currently in state closed
2021-02-18 00:05:13.790849 (MainThread): SQL status: SUCCESS 1 in 1.04 seconds
2021-02-18 00:05:13.790849 (MainThread): On master: COMMIT
2021-02-18 00:05:13.790849 (MainThread): Using snowflake connection "master".
2021-02-18 00:05:13.790849 (MainThread): On master: COMMIT
2021-02-18 00:05:14.135141 (MainThread): SQL status: SUCCESS 1 in 0.34 seconds
2021-02-18 00:05:14.136138 (MainThread): On master: Close
2021-02-18 00:05:14.412341 (MainThread): 18:05:14 | 
2021-02-18 00:05:14.413373 (MainThread): 18:05:14 | Finished running 1 view model in 10.05s.
2021-02-18 00:05:14.413373 (MainThread): Connection 'master' was properly closed.
2021-02-18 00:05:14.414336 (MainThread): Connection 'list_DBT_POC' was properly closed.
2021-02-18 00:05:14.414336 (MainThread): Connection 'list_DBT_POC_dbt_jkolpak' was properly closed.
2021-02-18 00:05:14.414336 (MainThread): Connection 'model.dbtpoc.model' was properly closed.
2021-02-18 00:05:14.420320 (MainThread): 
2021-02-18 00:05:14.433967 (MainThread): Completed successfully
2021-02-18 00:05:14.434964 (MainThread): 
Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
2021-02-18 00:05:14.435964 (MainThread): Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017FEC98BD08>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017FEA3EB2C8>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000017FEC6753C8>]}
2021-02-18 00:05:14.435964 (MainThread): Flushing usage events
